<section id="annotare-help">
  <div class="small-12 medium-9 columns">
    
    <h2>Accepted raw microarray files formats</h2>
    <p><strong>Important notes on file preparation:</strong></p>
    <ol>
        <li>Do not compress your microarray data files.</li>
        <li>Submit raw or raw matrix file(s) for every sample/hybridisation of your experiment.</li>
        <li>Make sure the file names are constructed <em>only</em> from alphanumerals [A-Z,a-z,0-9], underscores [_] and
        dots [.], with no whitespaces, brackets, other punctuations or symbols.</li>
        <li>Any spreadsheet/matrix file should be saved in tab-delimited text (<code>*.txt</code>) format and not Excel
        format (<code>*.xls</code>, <code>*.xlsx</code>)
    </ol>
    <h3>What are "raw" files?</h3>
    <ul>
        <li><strong>Raw (recommended)</strong>: The "native" files generated by the microarray scanner software.
        Make sure you do not change/edit the native files in any way, and submit one raw file per hybridisation
        assay. (One assay can consist of just one channel, as in Affymetrix experiments, or two channels, as in spotted
        arrays often with red and green channels from two different dyes/fluorophores.)<br><br>
        Commercial microarray manufacturers have developed different raw data file formats over the years.
        If you are unsure about whether your raw files are in an accepted format, please check
        <a href="#accepted_format_by_platform">the list below</a>.</li><br>
        <li><strong>Raw Matrix</strong>: A raw file in <em>tab-delimited text (.txt)</em> format*,
        that contains data from more than one hybridisation assay (probes in rows and assays in columns). The format
        requirements are strict (except for Illumina GenomeStudio data files). See
        <a href="strict_mtab_matrix.html">matrix guidelines and examples</a>.</li>
    </ul>

    <a id="accepted_format_by_platform"></a><h3>Accepted formats by platform</h3>

    <p>ArrayExpress submission system recognises the raw data file platform using column headings in the file's header:</p>

    <ul><li>Common platforms:</li>
        <ul>
            <li><a href="#affymetrix">Affymetrix</a></li>
            <li><a href="#agilent">Agilent</a></li>
            <li><a href="#illumina_raw">Illumina</a></li>
            <li><a href="#genepix">GenePix</a></li>
            <li><a href="#nimblescan">NimbleScan</a></li>

        </ul>
        <li>Others:</li>
        <ul>
            <li><a href="#scanalyze">ScanAlyze</a></li>
            <li><a href="#scanarray">ScanArray</a></li>
            <li><a href="#scanarray">QuantArray</a></li>
            <li><a href="#arrayvision">Arrayvision</a></li>
            <li><a href="#spotfinder">Spotfinder</a></li>
            <li><a href="#bluefuse">BlueFuse</a></li>
            <li><a href="#ucsfspot">UCSF Spot</a></li>
            <li><a href="#appliedbiosystems">Applied Biosystems</a></li>
            <li><a href="#codelink">CodeLink</a></li>
            <li><a href="#imagene">Imagene</a></li>
            <li><a href="#csirospot">CSIRO Spot</a></li>
            <li><a href="#generic">Generic (for spotted arrays, non-platform specific)</a></li>
        </ul>
    </ul>
    <br>
    <h4>
        <a id="affymetrix">Affymetrix</a>
    </h4>
    <p>
        Our submission system recognises .CEL and .EXP files using both the old GDAC
        formats and the newer GCOS/XDA formats.
    </p>

    <h4>
        <a id="agilent">Agilent</a>
    </h4>
    <p>A file containing these headings is recognised as an Agilent format file:</p>

    <table>
        <tr>
            <td>Row</td>
            <td>Col</td>
            <td>PositionX</td>
            <td>PositionY</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="illumina_raw">Illumina</a>
    </h4>
    <p>Illumina raw data files are usually either in plain text or binary format. Plain text files are generated by the <a href="https://www.illumina.com/techniques/microarrays/array-data-analysis-experimental-design/genomestudio.html">Illumina GenomeStudio software </a>. The binary "IDAT" files (stands for "intensity data file") are generated by the scanner and can be parsed using R/BioConductor packages such as <a href="http://www.bioconductor.org/packages/release/bioc/html/illuminaio.html">illuminaio</a>). IDAT is the preferred file format, as it is a binary format, containing all information required to analyse the data. In contrast, plain-text files can be missing information such as which are the control probes; this is sometimes provided in a separate file, but not always, and heterogeneity in raw data file formats makes systematic analysis of data difficult. Another disadvantage of plain-text files is that they are susceptible to human-introduced errors, as it is easy for someone to open the file in a text editor or spreadsheet program and accidentally change its content.
        If you're submitting a GenomeStudio text file, below is an example of the expected column headings:</p>

    <table>
        <tr>
            <td>PROBE_ID</td>
            <td>Assay_Name_1.QT1</td>
            <td>Assay_Name_1.QT2</td>
            <td>Assay_Name_2.QT1</td>
            <td>Assay_Name_2.QT2</td>
        </tr>
    </table>
    <p>PROBE_IDs are always in the format of "ILMN_123456".  Assay Names are automatically generated by Annotare
        and can be found in the <code>SDRF Preview</code> tab while you are preparing your submission. <code>QT</code> stands for
        quantitation type, i.e. the type of measurement recorded in the column, e.g. <code>AVGSignal</code>.
        You can have as many quantitation types as required. Order the columns by sample names, then by quantitation types.</p>



    <h4>
        <a id="genepix">GenePix</a>
    </h4>
    <p>GenePix format files (usually with file extension .gpr or .txt) are recognised using the following column headings:</p>

    <table>
        <tr>
            <td>Block</td>
            <td>Column</td>
            <td>Row</td>
            <td>X</td>
            <td>Y</td>
        </tr>
    </table>

    <h4>
        <a id="nimblescan">NimbleScan</a>
    </h4>
    <p>NimbleScan files (Feature, Probe and Pair) all contain the following headings:</p>

    <table>
        <tr>
            <td>PROBE_ID</td>
            <td>X</td>
            <td>Y</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>


    <h4>
        <a id="scanalyze">ScanAlyze</a>
    </h4>
    <p>The following column headings are recognised as being from a ScanAlyze format
        file:</p>

    <table>
        <tr>
            <td>GRID</td>
            <td>COL</td>
            <td>ROW</td>
            <td>LEFT</td>
            <td>TOP</td>
            <td>RIGHT</td>
            <td>BOT</td>
        </tr>
    </table>

    <h4>
        <a id="scanarray">ScanArray/QuantArray</a>
    </h4>
    <p>ScanArray Express files are recognised from the following headings:</p>

    <table>
        <tr>
            <td>Array Column</td>
            <td>Array Row</td>
            <td>Spot Column</td>
            <td>Spot Row</td>
            <td>X</td>
            <td>Y</td>
        </tr>
    </table>

    <p>while the older QuantArray format has these headings:</p>

    <table>
        <tr>
            <td>Array Column</td>
            <td>Array Row</td>
            <td>Column</td>
            <td>Row</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="arrayvision">ArrayVision</a>
    </h4>
    <p>The following column headings are recognised as indicating an ArrayVision
        format file:</p>

    <table>
        <tr>
            <td>Primary</td>
            <td>Secondary</td>
        </tr>
    </table>

    <p>Newer "lg2" ArrayVision files are identified by the following column headings:</p>

    <table>
        <tr>
            <td>Spot labels</td>
        </tr>
    </table>

    <h4>
        <a id="spotfinder">Spotfinder</a>
    </h4>
    <p>Spotfinder files are recognised by the following column headings:</p>

    <table>
        <tr>
            <td>MC</td>
            <td>MR</td>
            <td>SC</td>
            <td>SR</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="bluefuse">BlueFuse</a>
    </h4>
    <p>A file containing the following headings is recognised as a BlueFuse file:</p>

    <table>
        <tr>
            <td>COL</td>
            <td>ROW</td>
            <td>SUBGRIDCOL</td>
            <td>SUBGRIDROW</td>
        </tr>
    </table>

    <h4>
        <a id="ucsfspot">UCSF Spot</a>
    </h4>
    <p>UCSF Spot files are recognised by the following column headings:</p>

    <table>
        <tr>
            <td>Arr-colx</td>
            <td>Arr-coly</td>
            <td>Spot-colx</td>
            <td>Spot-coly</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="appliedbiosystems">Applied Biosystems</a>
    </h4>
    <p>Files generated by Applied Biosystems software have the following headings:</p>

    <table>
        <tr>
            <td>Probe_ID</td>
            <td>Gene_ID</td>
        </tr>
    </table>

    <h4>
        <a id="codelink">CodeLink</a>
    </h4>
    <p>CodeLink Expression Analysis files are identified using the following:</p>

    <table>
        <tr>
            <td>Logical_row</td>
            <td>Logical_col</td>
            <td>Center_X</td>
            <td>Center_Y</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="imagene">ImaGene</a>
    </h4>
    <p>ImaGene files are recognised using the following columns:</p>

    <table>
        <tr>
            <td>Meta Column</td>
            <td>Meta Row</td>
            <td>Column</td>
            <td>Row</td>
            <td>Field</td>
            <td>Gene ID</td>
        </tr>
    </table>

    <p>The ImaGene 3.0 format is also supported:</p>

    <table>
        <tr>
            <td>Meta_col</td>
            <td>Meta_row</td>
            <td>Sub_col</td>
            <td>Sub_row</td>
            <td>Name</td>
            <td>Selected</td>
        </tr>
    </table>

    <h4>
        <a id="csirospot">CSIRO Spot</a>
    </h4>
    <p>CSIRO Spot files contain the following columns:</p>

    <table>
        <tr>
            <td>grid_c</td>
            <td>grid_r</td>
            <td>spot_c</td>
            <td>spot_r</td>
            <td>indexs</td>
        </tr>
    </table>

    <h4>
        <a id="generic">Generic (for spotted arrays, non-platform specific)</a>
    </h4>
    <p>If your raw data file contains BlockColumn/BlockRow/Column/Row fields denoting
    <a href="/arrayexpress/help/creating_an_adf.html#Concepts">probe location on a spotted array</a>, you can use this
    generic format with four columns followed by columns of data:</p>

    <table>
        <tr>
            <td>MetaColumn</td>
            <td>MetaRow</td>
            <td>Column</td>
            <td>Row</td>
        </tr>
    </table>
    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>
  </div>
</section>

  <div class="medium-3 columns sidebar">
    <ul class="vertical menu accordion-menu" data-accordion-menu>
      <li><a href="/fg/annotare/help/" class="active" >Submission guide</a></li>
      <li><a>Before you start</a>
        <ul class="menu vertical nested">
          <li><a href="what_to_submit.html">Submission overview</a></li>
          <li class="menu vertical nested is-active"><a href="">Accepted file formats</a>
            <ul>
              <li><a class="active" href="accepted_raw_ma_file_formats.html">Microarray raw data</a></li>
              <li><a href="accepted_processed_ma_file_formats.html">Microarray processed data</a></li>
              <li><a href="UHTS_submissions.html#HowToSubmit">Sequencing raw data</a></li>
              <li><a href="UHTS_submissions.html#processed_data">Sequencing processed data</a></li>
            </ul>
          </li>
          <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
          <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
        </ul>
      </li>
      <li><a>Annotare step-by-step</a>
        <ul class="menu vertical nested">
          <li><a href="getting_started.html">Getting started</a></li>
          <li><a href="time_saving_features.html">Time-saving features</a></li>
          <li><a href="describe_exp.html">Describe experiment</a></li>
          <li><a href="sample_attributes.html">Sample attributes and protocols</a></li>
          <li><a href="">Technology-specific details</a>
            <ul>
              <li><a href="seq_lib_spec.html">Sequencing library information</a></li>
              <li><a href="singlecell_submission_guide.html">Single cell information</a></li>
              <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
          </li>
          <li><a href="">Upload and assign files</a>
            <ul>
              <li><a href="file_upload.html#direct">Direct upload</a></li>
              <li><a href="ftp_upload.html">FTP/Aspera upload</a></li>
              <li><a href="file_upload.html#ftp">Import FTP/Aspera uploaded files</a></li>
              <li><a href="assign_files.html">Assign files</a></li>
            </ul>
          </li>
          <li><a href="">Validate and submit</a>
            <ul>
              <li><a href="validate_exp.html#validate">What is validation?</a></li>
              <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
              <li><a href="submit_exp.html#save_sub">Saving a copy of your submission</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="">After submission</a>
        <ul>
          <li><a href="after_sub_private_access.html">Access private experiments</a></li>
          <li><a href="after_sub_modify_data.html">Modify or remove an existing experiment</a></li>
          <li><a href="after_sub_data_availability.html">Data availability and release policy</a></li>
        </ul>
      </li>
    </ul>
  </div> <!-- /medium-3 -->
