<section id="annotare-help">
  <div class="small-12 medium-9 columns">
    
    <h2>Accepted processed microarray files formats</h2>
    <p><strong>Important notes on file preparation:</strong></p>
    <ol>
        <li>Do not compress your microarray data files.</li>
        <li>Make sure the file names are constructed <em>only</em> from alphanumerals [A-Z,a-z,0-9], underscores
        [_] and dots [.], with no whitespaces, brackets, other punctuations or symbols.</li>
        <li>Any spreadsheet/matrix file should be saved in tab-delimited text (<code>*.txt</code>) format and not
        Excel format (<code>*.xls</code>, <code>*.xlsx</code>). If you're unfamiliar with this format, please see
        <a href="https://wiki.openoffice.org/wiki/Documentation/FAQ/Calc/Files/How_do_I_open_a_tab-delimited_file_in_OpenOffice.org_Spreadsheet%3F_What_if_I_have_a_different_type_of_delimiter%3F">OpenOffice Calc</a>
        or <a href="http://office.microsoft.com/en-gb/excel-help/import-or-export-text-txt-or-csv-files-HP010342598.aspx">Microsoft Excel</a> guide.</li>
    </ol>

    <h3>What are "processed" files?</h3>

    <p>Processed files are generated from raw files by procedures such as background correction, normalisation, and
    further statistical analyses (e.g. calculating fold-changes and associated p-values). We accept either "native"
    processed files from microarray scanner software (e.g. ".chp" files from Affymetrix scanners, output files from
    GenomeStudio software for Illumina BeadChip), or two-dimensional spreadsheet files in
    <strong>tab-delimited text (.txt)</strong> format.  For the latter, the probes/probesets/gene names are in rows,
    and data from one or more hybridisations are in columns. We accept processed files from the following scenarios:
    <ul>
        <li>one processed file per hybridisation, i.e. you have a series of processed files;</li>
        <li>one spreadsheet ("matrix") file containing normalised data from all hybridisations;</li>
        <li>several spreadsheet ("matrix") files containing normalised data from different stages of data processing,
        e.g. one file containing normalised probe intensities and another containing fold-change data summarised at
        the gene level.</li>
    </ul>


    <h3>What should a processed text file look like?</a></h3>
    <p>In the two-dimensional table, you should have probes/genes in rows and samples/data in columns:</p>
    <ul>
        <li><strong>Probes/genes in rows</strong>:  Where possible, as row headers, you should use official probe
        names/identifiers, matching those in the <a href="/arrayexpress/help/adf_submissions_overview.html">array design
        file</a>, so one can map each row of data to the correct probe. Put the probe identifiers in the first column
        under a heading <code>Reporter Identifier</code> (for probes) or <code>CompositeSequence Identifier </code>
        (for "composite" collation of probes, most common example being Affymetrix probe sets). If probe identifiers are
        not available, try to use proper gene symbols or other identifiers (e.g. GenBank cDNA accession, UniProt protein
        accession).</li>

        <li><strong>Samples/Data in columns</strong>: Where possible, label each data column with the same sample names
        as you declare on the Annotare forms, or use terms as they appear in your manuscript. This would allow mapping
        of a column of data to correct sample(s).</li>
    </ul>
    <p>A processed .txt file containing data from one single hybridisation should look like this:</p>
    <table>
        <tr>
            <td><strong>Reporter Identifier<strong></td>
            <td><strong>sample 1 normalised intensity</strong></td>
            <td><strong>sample 1 background</strong></td>
        </tr>
        <tr>
            <td><strong>probe_name_1</strong></td>
            <td><i>233.5</i></td>
            <td><i>69.1</i></td>
        </tr>
        <tr>
            <td><strong>probe_name_2</strong></td>
            <td><i>129.4</i></td>
            <td><i>27.6</i></td>
        </tr>
    </table>
    <p>And here is an example where gene names are used as row headings:</p>
    <table>
        <tr>
            <td><strong>Human HGNC gene name</strong></td>
            <td><strong>sample 1 normalised intensity</strong></td>
            <td><strong>sample 1 background</strong></td>
        </tr>
        <tr>
            <td><strong>CDKN2A</strong></td>
            <td><i>233.5</i></td>
            <td><i>69.1</i></td>
        </tr>
        <tr>
            <td><strong>BRCA2</strong></td>
            <td><i>129.4</i></td>
            <td><i>27.6</i></td>
        </tr>
    </table>
    <p>&nbsp;</p>
    <p>Processed "matrices" summarising data from multiple hybridisations should look like the following. Again, as for
    per-hybridisation processed files, if probe identifiers are not available, try to use proper gene symbols or other
    identifiers (e.g. GenBank cDNA accession, UniProt protein accession).</p>
    <em>Matrix of normalised values per sample:</em><br><br>

    <table>
        <tr>
            <td><strong>Reporter Identifier</strong></td>
            <td><strong>sample 1 normalised</strong></td>
            <td><strong>sample 2 normalised</strong></td>
            <td><strong>sample 3 normalised</strong></td>
            <td><strong>sample 4 normalised</strong></td>
        </tr>
        <tr>
            <td><strong>probe_name_1</strong></td>
            <td><i>26.9</i></td>
            <td><i>44.3</i></td>
            <td><i>62.3</i></td>
            <td><i>58.5</i></td>
        </tr>
        <tr>
            <td><strong>probe_name_2</strong></td>
            <td><i>22.9</i></td>
            <td><i>43.7</i></td>
            <td><i>58.2</i></td>
            <td><i>67.4</i></td>
        </tr>
    </table>
    <p>&nbsp;</p>
    <table>
        <tr>
            <td><strong>GenBank accession</strong></td>
            <td><strong>sample 1 normalised</strong></td>
            <td><strong>sample 2 normalised</strong></td>
            <td><strong>sample 3 normalised</strong></td>
            <td><strong>sample 4 normalised</strong></td>
        </tr>
        <tr>
            <td><strong>BC000578</strong></td>
            <td><i>26.9</i></td>
            <td><i>44.3</i></td>
            <td><i>62.3</i></td>
            <td><i>58.5</i></td>
        </tr>
        <tr>
            <td><strong>M31642</strong></td>
            <td><i>22.9</i></td>
            <td><i>43.7</i></td>
            <td><i>58.2</i></td>
            <td><i>67.4</i></td>
        </tr>
    </table>
    <p>&nbsp;</p>
    <em>Matrix of summarised values (one column of data maps to multiple samples):</em><br><br>
    <table>
        <tr>
            <td><strong>Reporter Identifier</strong></td>
            <td><strong>drug A treated average</strong></i></td>
            <td><strong>drug B treated average</strong></i></td>
            <td><strong>untreated control average</strong></i></td>
        </tr>
        <tr>
            <td><strong>probe_name_1</strong></td>
            <td><i>44.6</i></td>
            <td><i>89.3</i></td>
            <td><i>290.15</i></td>
        </tr>
        <tr>
            <td><strong>probe_name_2</strong></td>
            <td><i>98.3</i></td>
            <td><i>36.7</i></td>
            <td><i>100.52</i></td>
        </tr>
    </table>
    <p>&nbsp;</p>
    <h3>Processed matrix files in strict MAGE-TAB format (for advanced users)</a></h3>
    <p>For submitters who are familiar with <a href="http://www.fged.org/projects/mage-tab/" target="_blank">MAGE-TAB specification</a>,
    we also accept matrix files in strict MAGE-TAB format, which allows each data point in the file (in a given row and
    a given column) to be mapped to a particular assay in the experiment and to a particular probe/probe set in the
    <a href="/arrayexpress/help/adf_submissions_overview.html">array design file</a> in a human readable way and also
    programmatically. Check out this <a href="strict_mtab_matrix.html">guide on the strict matrix format</a> for more
    information.</p>
  </div>

  <!-- side navigation -->
  <div class="medium-3 columns sidebar">
    <ul class="vertical menu accordion-menu" data-accordion-menu>
      <li><a href="/fg/annotare/help/" class="active" >Submission guide</a></li>
      <li><a>Before you start</a>
        <ul class="menu vertical nested">
          <li><a href="what_to_submit.html">Submission overview</a></li>
          <li class="menu vertical nested is-active"><a href="">Accepted file formats</a>
            <ul>
              <li><a href="accepted_raw_ma_file_formats.html">Microarray raw data</a></li>
              <li><a class="active" href="accepted_processed_ma_file_formats.html">Microarray processed data</a></li>
              <li><a href="UHTS_submissions.html#HowToSubmit">Sequencing raw data</a></li>
              <li><a href="UHTS_submissions.html#processed_data">Sequencing processed data</a></li>
            </ul>
          </li>
          <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
          <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
        </ul>
      </li>
      <li><a>Annotare step-by-step</a>
        <ul class="menu vertical nested">
          <li><a href="getting_started.html">Getting started</a></li>
          <li><a href="time_saving_features.html">Time-saving features</a></li>
          <li><a href="describe_exp.html">Describe experiment</a></li>
          <li><a href="sample_attributes.html">Sample attributes and protocols</a></li>
          <li><a href="">Technology-specific details</a>
            <ul>
              <li><a href="seq_lib_spec.html">Sequencing library information</a></li>
              <li><a href="singlecell_submission_guide.html">Single cell information</a></li>
              <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
          </li>
          <li><a href="">Upload and assign files</a>
            <ul>
              <li><a href="file_upload.html#direct">Direct upload</a></li>
              <li><a href="ftp_upload.html">FTP/Aspera upload</a></li>
              <li><a href="file_upload.html#ftp">Import FTP/Aspera uploaded files</a></li>
              <li><a href="assign_files.html">Assign files</a></li>
            </ul>
          </li>
          <li><a href="">Validate and submit</a>
            <ul>
              <li><a href="validate_exp.html#validate">What is validation?</a></li>
              <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
              <li><a href="submit_exp.html#save_sub">Saving a copy of your submission</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="">After submission</a>
        <ul>
          <li><a href="after_sub_private_access.html">Access private experiments</a></li>
          <li><a href="after_sub_modify_data.html">Modify or remove an existing experiment</a></li>
          <li><a href="after_sub_data_availability.html">Data availability and release policy</a></li>
        </ul>
      </li>
    </ul>
  </div> <!-- /medium-3 -->
</section>
