<section id="annotare-help" class="grid_18 alpha">
    <h2>Accepted raw microarray files formats</h2>
    <p><strong>Important notes on file preparation:</strong></p>
    <ol>
        <li>Do not compress your microarray data files.</li>
        <li>Submit raw or raw matrix file(s) for every sample/hybridisation of your experiment.</li>
        <li>Make sure the file names are constructed <em>only</em> from alphanumerals [A-Z,a-z,0-9], underscores [_] and
        dots [.], with no whitespaces, brackets, other punctuations or symbols.</li>
        <li>Any spreadsheet/matrix file should be saved in tab-delimited text (<code>*.txt</code>) format and not Excel
        format (<code>*.xls</code>, <code>*.xlsx</code>)
    </ol>
    <h3>What are "raw" files?</h3>
    <ul>
        <li><strong>Raw (recommended)</strong>: The "native" files generated by the microarray scanner software.
        Make sure you do not change/edit the native files in any way, and submit one raw file per hybridisation
        assay. (One assay can consist of just one channel, as in Affymetrix experiments, or two channels, as in spotted
        arrays often with red and green channels from two different dyes/fluorophores.)<br><br>
        Commercial microarray manufacturers have developed different raw data file formats over the years.
        If you are unsure about whether your raw files are in an accepted format, please check
        <a href="#accepted_format_by_platform">the list below</a>.</li><br>
        <li><strong>Raw Matrix</strong>: A raw file in <em>tab-delimited text (.txt)</em> format*,
        that contains data from more than one hybridisation assay (probes in rows and assays in columns). The format
        requirements are strict (except for Illumina GenomeStudio data files). See
        <a href="strict_mtab_matrix.html">matrix guidelines and examples</a>.</li>
    </ul>

    <a id="accepted_format_by_platform"></a><h3>Accepted formats by platform</h3>

    <p>ArrayExpress submission system recognises the raw data file platform using column headings in the file's header:</p>

    <ul><li>Common platforms:</li>
        <ul>
            <li><a href="#affymetrix">Affymetrix</a></li>
            <li><a href="#agilent">Agilent</a></li>
            <li><a href="#illumina_raw">Illumina</a></li>
            <li><a href="#genepix">GenePix</a></li>
            <li><a href="#nimblescan">NimbleScan</a></li>

        </ul>
        <li>Others:</li>
        <ul>
            <li><a href="#scanalyze">ScanAlyze</a></li>
            <li><a href="#scanarray">ScanArray</a></li>
            <li><a href="#scanarray">QuantArray</a></li>
            <li><a href="#arrayvision">Arrayvision</a></li>
            <li><a href="#spotfinder">Spotfinder</a></li>
            <li><a href="#bluefuse">BlueFuse</a></li>
            <li><a href="#ucsfspot">UCSF Spot</a></li>
            <li><a href="#appliedbiosystems">Applied Biosystems</a></li>
            <li><a href="#codelink">CodeLink</a></li>
            <li><a href="#imagene">Imagene</a></li>
            <li><a href="#csirospot">CSIRO Spot</a></li>
            <li><a href="#generic">Generic (for spotted arrays, non-platform specific)</a></li>
        </ul>
    </ul>

    <h4>
        <a id="affymetrix">Affymetrix</a>
    </h4>
    <p>
        Our submission system recognises .CEL and .EXP files using both the old GDAC
        formats and the newer GCOS/XDA formats.
    </p>

    <h4>
        <a id="agilent">Agilent</a>
    </h4>
    <p>A file containing these headings is recognized as an Agilent format file:</p>

    <table>
        <tr>
            <td>Row</td>
            <td>Col</td>
            <td>PositionX</td>
            <td>PositionY</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="illumina_raw">Illumina</a>
    </h4>
    <p>Several different types of files can be generated from Illumina arrays. We accept either
        text files generated by <a href="http://www.illumina.com/software/genomestudio_software.ilmn">
            Illumina GenomeStudio software</a> (previously known as the "BeadStudio" software,
        using the <a href="http://www.illumina.com/software/illumina_connect.ilmn#gene_expression_plug_ins">
            ArrayExpress Data Submission Report Plug-in</a>), or binary IDAT files ("intensity data files"),
        typically created if you are using R/BioConductor packages such as <a href="http://www.bioconductor.org/packages/release/bioc/html/illuminaio.html">illuminaio</a>. IDAT is the preferred format as it contains all information required for data analysis (e.g. control probes) and is less prone to file tampering (e.g. opening the raw file in a text editor and editing the content).
        If you're submitting a GenomeStudio text file, below is an example of the expected column headings:</p>

    <table>
        <tr>
            <td>PROBE_ID</td>
            <td>Assay_Name_1.QT1</td>
            <td>Assay_Name_1.QT2</td>
            <td>Assay_Name_2.QT1</td>
            <td>Assay_Name_2.QT2</td>
        </tr>
    </table>
    <p>PROBE_IDs are always in the format of "ILMN_123456".  Assay Names are automatically generated by Annotare
        and can be found in the <code>SDRF Preview</code> tab while you are preparing your submission. <code>QT</code> stands for
        quantitation type, i.e. the type of measurement recorded in the column, e.g. <code>AVGSignal</code>.
        You can have as many quantitation types as required. Order the columns by sample names, then by quantitation types.</p>



    <h4>
        <a id="genepix">GenePix</a>
    </h4>
    <p>GenePix format files are recognized using the following column headings:</p>

    <table>
        <tr>
            <td>Block</td>
            <td>Column</td>
            <td>Row</td>
            <td>X</td>
            <td>Y</td>
        </tr>
    </table>

    <h4>
        <a id="nimblescan">NimbleScan</a>
    </h4>
    <p>NimbleScan files (Feature, Probe and Pair) all contain the following headings:</p>

    <table>
        <tr>
            <td>PROBE_ID</td>
            <td>X</td>
            <td>Y</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>


    <h4>
        <a id="scanalyze">ScanAlyze</a>
    </h4>
    <p>The following column headings are recognized as being from a ScanAlyze format
        file:</p>

    <table>
        <tr>
            <td>GRID</td>
            <td>COL</td>
            <td>ROW</td>
            <td>LEFT</td>
            <td>TOP</td>
            <td>RIGHT</td>
            <td>BOT</td>
        </tr>
    </table>

    <h4>
        <a id="scanarray">ScanArray/QuantArray</a>
    </h4>
    <p>ScanArray Express files are recognized from the following headings:</p>

    <table>
        <tr>
            <td>Array Column</td>
            <td>Array Row</td>
            <td>Spot Column</td>
            <td>Spot Row</td>
            <td>X</td>
            <td>Y</td>
        </tr>
    </table>

    <p>while the older QuantArray format has these headings:</p>

    <table>
        <tr>
            <td>Array Column</td>
            <td>Array Row</td>
            <td>Column</td>
            <td>Row</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="arrayvision">ArrayVision</a>
    </h4>
    <p>The following column headings are recognized as indicating an ArrayVision
        format file:</p>

    <table>
        <tr>
            <td>Primary</td>
            <td>Secondary</td>
        </tr>
    </table>

    <p>Newer "lg2" ArrayVision files are identified by the following column headings:</p>

    <table>
        <tr>
            <td>Spot labels</td>
        </tr>
    </table>

    <h4>
        <a id="spotfinder">Spotfinder</a>
    </h4>
    <p>Spotfinder files are recognized by the following column headings:</p>

    <table>
        <tr>
            <td>MC</td>
            <td>MR</td>
            <td>SC</td>
            <td>SR</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="bluefuse">BlueFuse</a>
    </h4>
    <p>A file containing the following headings is recognized as a BlueFuse file:</p>

    <table>
        <tr>
            <td>COL</td>
            <td>ROW</td>
            <td>SUBGRIDCOL</td>
            <td>SUBGRIDROW</td>
        </tr>
    </table>

    <h4>
        <a id="ucsfspot">UCSF Spot</a>
    </h4>
    <p>UCSF Spot files are recognized by the following column headings:</p>

    <table>
        <tr>
            <td>Arr-colx</td>
            <td>Arr-coly</td>
            <td>Spot-colx</td>
            <td>Spot-coly</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="appliedbiosystems">Applied Biosystems</a>
    </h4>
    <p>Files generated by Applied Biosystems software have the following headings:</p>

    <table>
        <tr>
            <td>Probe_ID</td>
            <td>Gene_ID</td>
        </tr>
    </table>

    <h4>
        <a id="codelink">CodeLink</a>
    </h4>
    <p>CodeLink Expression Analysis files are identified using the following:</p>

    <table>
        <tr>
            <td>Logical_row</td>
            <td>Logical_col</td>
            <td>Center_X</td>
            <td>Center_Y</td>
        </tr>
    </table>

    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>

    <h4>
        <a id="imagene">ImaGene</a>
    </h4>
    <p>ImaGene files are recognized using the following columns:</p>

    <table>
        <tr>
            <td>Meta Column</td>
            <td>Meta Row</td>
            <td>Column</td>
            <td>Row</td>
            <td>Field</td>
            <td>Gene ID</td>
        </tr>
    </table>

    <p>The ImaGene 3.0 format is also supported:</p>

    <table>
        <tr>
            <td>Meta_col</td>
            <td>Meta_row</td>
            <td>Sub_col</td>
            <td>Sub_row</td>
            <td>Name</td>
            <td>Selected</td>
        </tr>
    </table>

    <h4>
        <a id="csirospot">CSIRO Spot</a>
    </h4>
    <p>CSIRO Spot files contain the following columns:</p>

    <table>
        <tr>
            <td>grid_c</td>
            <td>grid_r</td>
            <td>spot_c</td>
            <td>spot_r</td>
            <td>indexs</td>
        </tr>
    </table>

    <h4>
        <a id="generic">Generic (for spotted arrays, non-platform specific)</a>
    </h4>
    <p>If your raw data file contains BlockColumn/BlockRow/Column/Row fields denoting
    <a href="/arrayexpress/help/creating_an_adf.html#Concepts">probe location on a spotted array</a>, you can use this
    generic format with four columns followed by columns of data:</p>

    <table>
        <tr>
            <td>MetaColumn</td>
            <td>MetaRow</td>
            <td>Column</td>
            <td>Row</td>
        </tr>
    </table>
    <p style="text-align: right;">
        <a href="#top" class="linktotop">Top</a>
    </p>
</section>

<aside class="grid_6 omega">
    <h3>Annotare Guide</h3>
    <ol id="secondary-nav">
        <li><h4>Getting started</h4>
            <ul>
                <li><a href="getting_started.html#register">Register for submission account</a></li>
                <li><a href="getting_started.html#start_new_sub">Start a new submission</a></li>
                <li><a href="getting_started.html#sub_status">Understand submission status</a></li>
            </ul>
        </li>
        <li><h4>Describe experiment</h4>
            <ul>
                <li><a href="describe_exp.html#expt_bg">Explain experiment background</a></li>
                <li><a href="describe_exp.html#release_date">Set reasonable release date</a></li>
                <li><a href="describe_exp.html#remain_anonymous">Enable anonymity</a></li>
                <li><a href="describe_exp.html#contact_citation">Provide contact details</a></li>
            </ul>
        </li>
        <li><h4>Time-saving features</h4>
            <ul>
                <li><a href="time_saving_features.html#fill_down_import">Fill-down and import values</a></li>
                <li><a href="time_saving_features.html#upload_multiple_files">Upload multiple files in one go</a></li>
            </ul>
        </li>
        <li><h4>Sample attributes</h4>
            <ul>
                <li><a href="sample_attributes.html#annotate">Annotate samples, nominate experimental variable(s)</a></li>
                <li><a href="sample_attributes.html#discrete_terms">Annotate with discrete terms</a></li>
                <li><a href="sample_attributes.html#protocols">Protocols</a></li>
            </ul>
        </li>
        <li><h4>Microarray experiment</h4>
            <ul>
                <li class="active"><a href="accepted_raw_ma_file_formats.html">Accepted raw file types</a></li>
                <li><a href="accepted_processed_ma_file_formats.html">Accepted processed file formats</a></li>
                <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
        </li>
        <li><h4>Sequencing experiment</h4>
            <ul>
                <li><a href="/arrayexpress/help/UHTS_submissions.html#DataTypes">Accepted data types</a></li>
                <li><a href="/arrayexpress/help/UHTS_submissions.html#HowToSubmit">How to prepare raw files</a></li>
                <li><a href="seq_lib_spec.html">Provide library information</a></li>
                <li><a href="/arrayexpress/help/UHTS_submissions.html#processed_data">Accepted processed file formats</a></li>
            </ul>
        </li>
        <li><h4>Upload files and assign to samples</h4>
            <ul>
                <li><a href="file_upload.html#direct">Direct upload</a></li>
                <li><a href="file_upload.html#ftp">FTP upload</a></li>
                <li><a href="assign_files.html">Assign files</a></li>
            </ul>
        </li>
        <li><h4>Validate and submit</h4>
            <ul>
                <li><a href="validate_exp.html#validate">What is validation?</a></li>
                <li><a href="validate_exp.html#fix_errors">How to fix validation errors?</a></li>
                <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
            </ul>
        </li>
    </ol>
</aside>
