<section id="annotare-help">
  <div class="small-12 medium-9 columns">
    
  <h2>List of allowed databases for reporter or composite element annotation</h2>

  <p>Select a name from <strong>DATABASE NAME</strong> below and insert it into the square brackets of <code>Reporter Database Entry [xxx]</code> or <code>Composite Element Database Entry</code> column headings in the <a href="creating_an_adf.html#ADFTable">ADF table</a>.<p>

  <table class="table">
    <thead>
      <tr><th>DATABASE NAME</th><th>DATABASE DETAILS</th><th>DATABASE URL</th></tr>
    </thead>
    <tbody>
      <tr><td>abrc_stock</td><td>Arabidopsis Biological Resource Center (ABRC) at The Ohio State University</td><td>http://www.arabidopsis.org/abrc/</td></tr>
      <tr><td>AmoebaDB</td><td>Amoeba DB</td><td>amoebadb.org/?</td></tr>
      <tr><td>asrp</td><td>Arabidopsis Small RNA Project</td><td>asrp.danforthcenter.org/?</td></tr>
      <tr><td>atcc</td><td>ATCC</td><td>http://www.lgcstandards-atcc.org</td></tr>
      <tr><td>bdgp</td><td>Berkeley Drosophila Genome Project</td><td>http://www.fruitfly.org</td></tr>
      <tr><td>beebase</td><td>Bee base</td><td>http://hymenopteragenome.org/beebase/</td></tr>
      <tr><td>candida_genome_db</td><td>Candida genome database</td><td>www.candidagenome.org/?</td></tr>
      <tr><td>candidadb</td><td>Candida database</td><td>genolist.pasteur.fr/CandidaDB/?</td></tr>
      <tr><td>catma</td><td>Complete Arabidopsis Transcriptome MicroArray (CATMA) project</td><td>www.catma.org/?</td></tr>
      <tr><td>celera_transcript</td><td>Celera transcript</td><td>http://www.pantherdb.org/panther/globalSearch.do</td></tr>
      <tr><td>cog</td><td>NCBI COG (Phylogenetic classification of proteins encoded in complete genomes)</td><td>http://www.ncbi.nlm.nih.gov/COG</td></tr>
      <tr><td>dbsnp</td><td>NCBI dbSNP</td><td>www.ncbi.nlm.nih.gov/SNP/?</td></tr>
      <tr><td>dfci_rgi</td><td>DFCI Rat Gene Index </td><td>http://compbio.dfci.harvard.edu/tgi/cgi-bin/tgi/gimain.pl?gudb=rat</td></tr>
      <tr><td>dfci_asgi</td><td>DFCI Atlantic Salmon Gene Index (AsGI) </td><td>http://compbio.dfci.harvard.edu/tgi/gi/asgi/searching/xpress_search.html</td></tr>
      <tr><td>dfci_atgi</td><td>DFCI Arabidopsis Gene Index </td><td>http://compbio.dfci.harvard.edu/cgi-bin/tgi/gimain.pl?gudb=arab</td></tr>
      <tr><td>dfci_mtgi</td><td>DFCI Medicago truncatula Gene Index (MtGI) </td><td>http://compbio.dfci.harvard.edu/tgi/gi/mtgi/searching/xpress_search.html</td></tr>
      <tr><td>dfci_ssgi</td><td>DFCI Porcine Gene Index (SsGI) </td><td>http://compbio.dfci.harvard.edu/tgi/gi/ssgi/searching/xpress_search.html</td></tr>
      <tr><td>dfci_stgi</td><td>DFCI Solanum tuberosum Gene Index (StGI) </td><td>http://compbio.dfci.harvard.edu/tgi/gi/stgi/searching/xpress_search.html</td></tr>
      <tr><td>dfci_zgi</td><td>DFCI Zebrafish Gene Index (ZGI)</td><td>http://compbio.dfci.harvard.edu/tgi/cgi-bin/tgi/gimain.pl?gudb=zfish</td></tr>
      <tr><td>dfci_zmgi</td><td>DFCI Maize Gene Index (ZGI)</td><td>http://compbio.dfci.harvard.edu/cgi-bin/tgi/gimain.pl?gudb=maize</td></tr>
      <tr><td>dictybase</td><td>dictyBase for Dictyostelium discoideum</td><td>dictybase.org/?</td></tr>
      <tr><td>embl</td><td>European Nucleotide Archive at EMBL-EBI</td><td>www.ebi.ac.uk/ena</td></tr>
      <tr><td>ens_fam_id</td><td>Ensembl gene family stable identifier</td><td>www.ensembl.org</td></tr>
      <tr><td>ens_gene_id</td><td>Ensembl gene stable identifier</td><td>www.ensembl.org</td></tr>
      <tr><td>ens_trscrpt_id</td><td>Ensembl transcript stable identifier</td><td>www.ensembl.org</td></tr>
      <tr><td>ensembl</td><td>Ensembl</td><td>www.ensembl.org</td></tr>
      <tr><td>entrez</td><td>NCBI Entrez</td><td>http://www.ncbi.nlm.nih.gov/entrez</td></tr>
      <tr><td>entrez_gene</td><td>NCBI Entrez gene identifier</td><td>http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene</td></tr>

      <tr><td>flybase</td><td>Flybase</td><td>www.flybase.org</td></tr>
      <tr><td>flybase_bt</td><td>Flybase (body parts)</td><td>www.flybase.org</td></tr>
      <tr><td>flybase_computed_gene</td><td>Flybase (computed gene)</td><td>www.flybase.org</td></tr>
      <tr><td>flybase_dv</td><td>Flybase (developmental stages)</td><td>www.flybase.org</td></tr>
      <tr><td>genbank</td><td>GenBank</td><td>www.ncbi.nlm.nih.gov/genbank</td></tr>
      <tr><td>genecards</td><td>Genecards</td><td>http://www.genecards.org</td></tr>
      <tr><td>genedb</td><td>Gene DB</td><td>http://www.genedb.org/Homepage</td></tr>
      <tr><td>genesnps</td><td>GeneSNPs</td><td>www.genome.utah.edu/genesnps/</td></tr>
      <tr><td>genolevures</td><td>Genomic Exploration of the Hemiascomycete Yeasts</td><td>http://cbi.labri.fr/Genolevures/</td></tr>
      <tr><td>go</td><td>Gene Ontology</td><td>http://www.geneontology.org</td></tr>
      <tr><td>gpcrdb</td><td>GPCRDB (information system for G protein-coupled receptors)</td><td>www.gpcr.org/7tm/</td></tr>
      <tr><td>gxd</td><td>Gene Expression Database at Jackson Laboratory (mouse data)</td><td>www.informatics.jax.org/expression.shtml?</td></tr>
      <tr><td>halolex</td><td>HaloLex (the information system for halophilic archaea)</td><td>http://www.webcitation.org/getfile?fileid=01b4e193eb9c261f676e242d43a243e918fe52b5</td></tr>
      <tr><td>HarvEST35</td><td>ESTs from HarvEST project</td><td>http://harvest-web.org/hweb/pickassy.wc</td></tr>
      <tr><td>hgmd</td><td>The Human Gene Mutation Database at the Institute of Medical Genetics in Cardiff</td><td>www.hgmd.org/?</td></tr>
      <tr><td>howdy</td><td>Human Organized Whole Genome Database</td><td>http://howdy.biosciencedbc.jp/HOWDY/top_en.pl</td></tr>
      <tr><td>hpa</td><td>The human protein atlas</td><td>http://www.proteinatlas.org/index.php</td></tr>
      <tr><td>hugo</td><td>Human Genome Organization (gene names)</td><td>www.genenames.org/?</td></tr>
      <tr><td>image</td><td>IMAGE consortium</td><td>www.imageconsortium.org/?</td></tr>
      <tr><td>incyte</td><td>Incyte Genomics</td><td>http://www.incyte.com</td></tr>
      <tr><td>insdc_locus_tag</td><td>INSDC locus tags</td><td>www.ebi.ac.uk/ena/about/locus_tag</td></tr>
      <tr><td>interpro</td><td>InterPro at EMBL-EBI</td><td>www.ebi.ac.uk/interpro/?</td></tr>
      <tr><td>kegg</td><td>KEGG pathway database</td><td>www.genome.jp/kegg/pathway.html?</td></tr>
      <tr><td>locus</td><td>NCBI Gene</td><td>http://www.ncbi.nlm.nih.gov/gene/</td></tr>
      <tr><td>mgc</td><td>NCBI Mammalian Gene Collection</td><td>mgc.nci.nih.gov/?</td></tr>
      <tr><td>mgd</td><td>Mouse Genome Database</td><td>www.nih.gov/science/models/mouse/resources/mgd.html?</td></tr>
      <tr><td>mips</td><td>The MIPS Mammalian Protein-Protein Interaction Database</td><td>http://mips.helmholtz-muenchen.de/proj/ppi/</td></tr>
      <tr><td>mirbase</td><td>microRNA database</td><td>www.mirbase.org</td></tr>
      <tr><td>MO</td><td>MGED Ontology</td><td>http://mged.sourceforge.net/ontologies/MGEDontology.php</td></tr>
      <tr><td>msu_rga</td><td>Rice Genome Annotation Project</td><td>http://rice.plantbiology.msu.edu/analyses_search_blast.shtml</td></tr>
      <tr><td>mtb</td><td>Mouse Tumor Biology (MTB) Database </td><td>www.informatics.jax.org/mtbwi?</td></tr>
      <tr><td>nasc</td><td>The European Arabidopsis Stock Centre</td><td>http://arabidopsis.info/CollectionInfo?id=19</td></tr>
      <tr><td>ncbi_clone_registry</td><td>NCBI Clone registry</td><td>www.ncbi.nlm.nih.gov/clone?</td></tr>
      <tr><td>ncbitax</td><td>NCBI Taxonomy database</td><td>http://www.ncbi.nlm.nih.gov/taxonomy</td></tr>
      <tr><td>netaffx</td><td>Affymetrix NetAffx</td><td>http://www.affymetrix.com/analysis/index.affx</td></tr>
      <tr><td>neurosporadb</td><td>Neurospora database at Broad Institute</td><td>http://www.broadinstitute.org/annotation/genome/neurospora/GeneIndex.html</td></tr>
      <tr><td>nia_nih</td><td>NIH National Institute of Ageing</td><td>http://www.nia.nih.gov/research/scientific-resources</td></tr>
      <tr><td>omim</td><td>Online Mendelian Inheritance in Men</td><td>http://www.omim.org</td></tr>
      <tr><td>pdb</td><td>Protein Data Bank</td><td>www.rcsb.org/</td></tr>
      <tr><td>pfam</td><td>Protein family</td><td>http://pfam.xfam.org/</td></tr>
      <tr><td>pharmgkb</td><td>PharmaGKB (pharacogenomics knowledgebase)</td><td>www.pharmgkb.org</td></tr>
      <tr><td>phyca11</td><td>Phytophthora capsici sequencing project</td><td>http://genome.jgi-psf.org/Phyca11/Phyca11.home.html</td></tr>
      <tr><td>pir</td><td>Protein Information Resource</td><td>pir.georgetown.edu</td></tr>
      <tr><td>plantgdb</td><td>Plant GDB (resources for comparative plant genomics)</td><td>www.plantgdb.org/?</td></tr>
      <tr><td>PlantOntology</td><td>Plant Ontology database</td><td>http://plantontology.org/</td></tr>
      <tr><td>plasmodb</td><td>Plasmodium Genomics Resource</td><td>www.plasmodb.org/?</td></tr>
      <tr><td>populusDB</td><td>Populus tremula x tremuloides genomic sequence database.</td><td>www.populus.db.umu.se</td></tr>
      <tr><td>prf</td><td>Synthetic compounds database</td><td>http://www.prf.or.jp/aboutdb-e.html</td></tr>
      <tr><td>prodom</td><td>ProDom (a comprehensive set of protein domain families automatically generated from the UniProt Knowledge Database)</td><td>prodom.prabi.fr/?</td></tr>
      <tr><td>prosite</td><td>Prosite, database of protein domains, families and functional sites</td><td>http://www.expasy.ch/prosite</td></tr>
      <tr><td>pubmed</td><td>PubMed literature database</td><td>http://www.ncbi.nlm.nih.gov/pubmed</td></tr>
      <tr><td>refseq</td><td>NCBI RefSeq</td><td>http://www.ncbi.nlm.nih.gov/refseq</td></tr>
      <tr><td>rgd</td><td>Rat Genome Database</td><td>rgd.mcw.edu/?</td></tr>
      <tr><td>rgd_qtl</td><td>Rat Genome Database QTLs</td><td>rgd.mcw.edu/?</td></tr>
      <tr><td>rhizobase</td><td>Rhizobase (soil and rhizosphere bacteria of agronomic importance)</td><td>genome.microbedb.jp/rhizobase/?</td></tr>
      <tr><td>scop</td><td>Structural classification of proteins</td><td>scop.mrc-lmb.cam.ac.uk/scop/?</td></tr>
      <tr><td>sgd</td><td>Saccharomyces Genome Database (SGD)</td><td>www.yeastgenome.org/</td></tr>
      <tr><td>sgn</td><td>Tomatoes genomics network</td><td>solgenomics.net/</td></tr>
      <tr><td>subtilist</td><td>Subtilist, Bacillus subtilis database</td><td>http://genolist.pasteur.fr/SubtiList/</td></tr>
      <tr><td>sulfolobus</td><td>Sulfolobus Database</td><td>www.sulfolobus.org/</td></tr>
      <tr><td>swissprot</td><td>UniProt/SwissProt (manually annotated and reviewed protein sequences)</td><td>www.ebi.ac.uk/uniprot</td></tr>
      <tr><td>tair</td><td>The Arabidopsis Information Resource.</td><td>www.arabidopsis.org/</td></tr>
      <tr><td>tair_anat</td><td>TAIR Anatomy Ontology.</td><td>www.arabidopsis.org/</td></tr>
      <tr><td>tair_dev</td><td>TAIR Development Ontology.</td><td>www.arabidopsis.org/</td></tr>
      <tr><td>tigr_tagi</td><td>TIGR: TaGI. The TIGR Anopheles gambiae (mosquito) Database.</td><td>www.arabidopsis.org/</td></tr>
      <tr><td>ToxoDB</td><td>ToxoDB: Toxoplasma genomics resources</td><td>http://www.toxodb.org</td></tr>
      <tr><td>toxoest</td><td>ToxoDB ESTs</td><td>http://toxodb.org/toxo/</td></tr>
      <tr><td>trembl</td><td>UniProt/trEMBL (automatically annotated protein sequences)</td><td>http://www.ebi.ac.uk/uniprot</td></tr>
      <tr><td>TriTrypDB</td><td>Database for pathogens of the family Trypanosomatidae, including organisms in both Leishmania and Trypanosoma genera</td><td>http://tritrypdb.org</td></tr>
      <tr><td>tuberculist</td><td>Genomic information on tubercle bacilli such as M. tuberculosis</td><td>tuberculist.epfl.ch/?</td></tr>
      <tr><td>uhn_cpg</td><td>UHN CpG Island Database</td><td>http://data.microarrays.ca/cpg/</td></tr>
      <tr><td>unigene</td><td>UniGene. Non-redundant, gene-oriented clusters</td><td>www.ncbi.nlm.nih.gov/unigene</td></tr>
      <tr><td>uniprot</td><td>UniProt</td><td>www.uniprot.org/?</td></tr>
      <tr><td>unists</td><td>NCBI UniSTS (now retired)</td><td>www.ncbi.nih.gov/genome/sts/</td></tr>
      <tr><td>vectorbase</td><td>VectorBase, for Invertebrate Vectors of Human Pathogens</td><td>https://www.vectorbase.org/?</td></tr>
      <tr><td>vega_cf</td><td>The Vertebrate Genome Annotation (VEGA) database (dog data)</td><td>vega.sanger.ac.uk/?</td></tr>
      <tr><td>vega_dr</td><td>The Vertebrate Genome Annotation (VEGA) database (zebrafish data)</td><td>vega.sanger.ac.uk/?</td></tr>
      <tr><td>vega_hs</td><td>The Vertebrate Genome Annotation (VEGA) database (human data)</td><td>vega.sanger.ac.uk/?</td></tr>
      <tr><td>vega_mm</td><td>The Vertebrate Genome Annotation (VEGA) database (mouse data)</td><td>vega.sanger.ac.uk/?</td></tr>
      <tr><td>wormbase</td><td>Wormbase, Genome and biology of C. elegans</td><td>www.wormbase.org</td></tr>
    </tbody>
  </table>
  </div>
  <!-- side navigation -->
  <div class="small-12 medium-3 columns sidebar">
    <ul class="vertical menu accordion-menu" data-accordion-menu>
      <li><a href="/fg/annotare/help/" class="active" >Submission guide</a></li>
      <li><a>Before you start</a>
        <ul class="menu vertical nested is-active">
          <li><a href="what_to_submit.html">Submission overview</a></li>
          <li><a href="">Accepted file formats</a>
            <ul>
              <li><a href="accepted_raw_ma_file_formats.html">Microarray raw data</a></li>
              <li><a href="accepted_processed_ma_file_formats.html">Microarray processed data</a></li>
              <li><a href="UHTS_submissions.html#HowToSubmit">Sequencing raw data</a></li>
              <li><a href="UHTS_submissions.html#processed_data">Sequencing processed data</a></li>
            </ul>
          </li>
          <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
          <li><a href="adf_submissions_overview.html">Array design submissions</a>
            <ul class="menu vertical nested is-active">
              <li><a href="adf_submissions_overview.html">Array design submission overview</a></li>
              <li><a href="creating_an_adf.html">Creating an ADF</a></li>
              <li><a class="active" href="adf_allowed_db_list.html">Allowed database names</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a>Annotare step-by-step</a>
        <ul class="menu vertical nested">
          <li><a href="getting_started.html">Getting started</a></li>
          <li><a href="time_saving_features.html">Time-saving features</a></li>
          <li><a href="describe_exp.html">Describe experiment</a></li>
          <li><a href="sample_attributes.html">Sample attributes and protocols</a></li>
          <li><a href="">Technology-specific details</a>
            <ul>
              <li><a href="seq_lib_spec.html">Sequencing library information</a></li>
              <li><a href="singlecell_submission_guide.html">Single cell information</a></li>
              <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
          </li>
          <li><a href="">Upload and assign files</a>
            <ul>
              <li><a href="file_upload.html#direct">Direct upload</a></li>
              <li><a href="ftp_upload.html">FTP/Aspera upload</a></li>
              <li><a href="file_upload.html#ftp">Import FTP/Aspera uploaded files</a></li>
              <li><a href="assign_files.html">Assign files</a></li>
            </ul>
          </li>
          <li><a href="">Validate and submit</a>
            <ul>
              <li><a href="validate_exp.html#validate">What is validation?</a></li>
              <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
              <li><a href="submit_exp.html#save_sub">Saving a copy of your submission</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="">After submission</a>
        <ul>
          <li><a href="after_sub_private_access.html">Access private experiments</a></li>
          <li><a href="after_sub_modify_data.html">Modify or remove an existing experiment</a></li>
          <li><a href="after_sub_data_availability.html">Data availability and release policy</a></li>
        </ul>
      </li>
    </ul>
  </div> <!-- /medium-3 -->

</section>
