<section id="annotare-help">
  <div class="small-12 medium-9 columns">
    
    <h2><a id="ADFSub"></a>Array design submissions</h2>

    <h3>What is an array design?</h3>

    <p>For a microarray experiment, an array design is the unique set of probes found on the microarray chip. In ArrayExpress, an ADF (Array Design Format) file captures information about a microarray chip. The ADF is a spreadsheet-like tab-delimited text file with some metadata header rows, followed by a multi-column table of probe information (here is <a href="creating_an_adf.html#WhatIsADF">an example</a>). Data from a microarray experiment cannot be interpreted properly if the array design used is unclear, especially when the microarray is custom-made for a particular research project.</p>

    <h3>Do I need to submit an array design for my experiment?</h3>
    <p>Many of the array designs, especially commercially available ones, are already accessioned in ArrayExpress as ADF files (one ADF for one microarray chip), in which case you do <em>not</em> need to submit any ADF to us. All you need to do is to specify the ADF by accession when asked by Annotare "What array did you use?".</p>
    <p> To find out whether your array design is already in ArrayExpress and get its accession, search by keywords (e.g. manufacturer name, species name) on the <a href="/arrayexpress/arrays/browse.html?directsub=on">ArrayExpress array design search page</a>. If the search does not return any results, include array designs we imported from NCBI Gene Expression Omnibus (GEO) to <a href="/arrayexpress/arrays/browse.html">broaden your search</a>.  Or if you know the NCBI GEO platform accession, you can convert it to the ArrayExpress equivalent by replacing "GPL" with "A-GEOD", e.g. "GPL1234" will be "A-GEOD-1234". If you cannot find a GEO platform accession in ArrayExpress, <a href="mailto:annotare@ebi.ac.uk">email us</a>, and we will import the array design.</p>

    <h3>How do I submit an array designs?</h3>

    h3><a id="templates"></a>1. Download an array design file (ADF) template</h3>

    <p>Pick a template that suits your microarray. (Check out this <a href="../assets/images/creating_an_adf.html#Concepts">concepts page</a> if you are unsure whether your microarray contains features, reporters or composite elements.)</p>
    <ul>
      <li>Commercially available arrays or custom arrays from commercial manufacturers (e.g. NimbleGen, Agilent, Affymetrix, Illumina), just metadata header to fill in, and send us any supporting files*: <a href="../assets/images/commercial_array_header_template.xls">commercial array design header template</a></li>

      <li>Custom spotted arrays with features in block row / block column / row / column grid coordinates: <a href="../assets/images/spotted_array_ADF_template.xls">spotted array template</a></li>

      <li>Custom arrays with only reporters (probes), without grid coordinate information: <a href="../assets/images/reporter_only_array_ADF_template.xls">reporter array template</a></li>

      <li>Custom arrays with reporters and composite elements: <a href="../assets/images/composite_element_array_ADF_template.xls">composite array template</a></li>
    </ul>

    <p><em>*Supporting files are those from the microarray's manufacturer containing information of each probe, e.g. NDF files for NimbleGen arrays, GAL files for spotted arrays.  An ArrayExpress curator will convert your supporting file into the <a href='creating_an_adf.html#WhatIsADF'>ADF table</a>, and append it to the metadata header you fill in to create a complete, valid ADF.</em></p>

    <p>&nbsp;</p>

    <h3>2. Fill in template with array design information</h3>

    <p>Refer to this <a href="creating_an_adf.html">ADF creation help page</a> for an explanation on how to fill in the template (e.g. what is mandatory, what are accepted values). Please stick to the guidelines so your ADF can be processed promptly. All the ADF header fields and ADF table column headings, highlighted in <strong>bold</strong>, are from a controlled vocabulary, so <em>please do not change them</em>.</p>

    <p>When you've finished filling in the template, please save the file in <strong>tab-delimited text (*.txt)</strong> format.  (If you're unsure about this format, this is how you do it with <a href="http://office.microsoft.com/en-gb/excel-help/import-or-export-text-txt-or-csv-files-HP010342598.aspx">Microsoft Excel</a> or <a href="https://wiki.openoffice.org/wiki/Documentation/FAQ/Calc/Files/How_do_I_open_a_tab-delimited_file_in_OpenOffice.org_Spreadsheet%3F_What_if_I_have_a_different_type_of_delimiter%3F">OpenOffice Calc</a>.)</p>

    <p>Below is an example snippet of a filled template (text in light blue filled in by the submitter):</p>

    <br>
    <img src="../assets/images/adf_snippet.png" alt="ADF_snippet" style='border:2px solid #385930'/>
    <br>

    <p>&nbsp;</p>

    <h3>3. Send the completed ADF file via FTP and then email us</h3>

    <p>Please send the ADF file by FTP/Aspera (<a href="ftp_upload.html">instructions here</a>).  If you are submitting a commercially available array design, don't forget to also send us the supporting files (e.g. NDF file for NimbleGen arrays, GAL file for spotted arrays). </p>

    <p>When the FTP/Aspera transfer is completed, please email us at <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> with the name of the transferred file(s).</p>

    <p>An ArrayExpress curator will get in touch by email once he/she has looked at your ADF file, including any questions or comments he/she may have. For commercial array designs, the curator will create the full ADF on your behalf by combining the meta-data header and some information converted from the annotation/support files.</p>
  </div>

  <!-- side navigation -->
  <div class="small-12 medium-3 columns sidebar">
    <ul class="vertical menu accordion-menu" data-accordion-menu>
      <li><a href="/fg/annotare/help/" class="active" >Submission guide</a></li>
      <li><a>Before you start</a>
        <ul class="menu vertical nested is-active">
          <li><a href="what_to_submit.html">Submission overview</a></li>
          <li><a href="">Accepted file formats</a>
            <ul>
              <li><a href="accepted_raw_ma_file_formats.html">Microarray raw data</a></li>
              <li><a href="accepted_processed_ma_file_formats.html">Microarray processed data</a></li>
              <li><a href="UHTS_submissions.html#HowToSubmit">Sequencing raw data</a></li>
              <li><a href="UHTS_submissions.html#processed_data">Sequencing processed data</a></li>
            </ul>
          </li>
          <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
          <li><a href="adf_submissions_overview.html">Array design submissions</a>
            <ul class="menu vertical nested is-active">
              <li><a class="active" href="adf_submissions_overview.html">Array design submission overview</a></li>
              <li><a href="creating_an_adf.html">Creating an ADF</a></li>
              <li><a href="adf_allowed_db_list.html">Allowed database names</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a>Annotare step-by-step</a>
        <ul class="menu vertical nested">
          <li><a href="getting_started.html">Getting started</a></li>
          <li><a href="time_saving_features.html">Time-saving features</a></li>
          <li><a href="describe_exp.html">Describe experiment</a></li>
          <li><a href="sample_attributes.html">Sample attributes and protocols</a></li>
          <li><a href="">Technology-specific details</a>
            <ul>
              <li><a href="seq_lib_spec.html">Sequencing library information</a></li>
              <li><a href="singlecell_submission_guide.html">Single cell information</a></li>
              <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
          </li>
          <li><a href="">Upload and assign files</a>
            <ul>
              <li><a href="file_upload.html#direct">Direct upload</a></li>
              <li><a href="ftp_upload.html">FTP/Aspera upload</a></li>
              <li><a href="file_upload.html#ftp">Import FTP/Aspera uploaded files</a></li>
              <li><a href="assign_files.html">Assign files</a></li>
            </ul>
          </li>
          <li><a href="">Validate and submit</a>
            <ul>
              <li><a href="validate_exp.html#validate">What is validation?</a></li>
              <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
              <li><a href="submit_exp.html#save_sub">Saving a copy of your submission</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="">After submission</a>
        <ul>
          <li><a href="after_sub_private_access.html">Access private experiments</a></li>
          <li><a href="after_sub_modify_data.html">Modify or remove an existing experiment</a></li>
          <li><a href="after_sub_data_availability.html">Data availability and release policy</a></li>
        </ul>
      </li>
    </ul>
  </div> <!-- /medium-3 -->

</section>
