<section id="annotare-help">
  <div class="small-12 medium-9 columns">

  <h2>Single-cell submission guide</h2>
  <br>
  <ol>
    <li><a href="#SampleInfo">Sample information</a>

    </li>
    <li><a href="#LibraryInfo">Library and sequencing information</a></li>
    <li><a href="#FileFormats">File formats</a></li>
    <li><a href="#Droplet">Droplet-based technologies</a></li>
      <li><a href="#Tips">A few more tips</a></li>
  </ol>
  <br>

  <p>Start your single-cell submission using the sequencing template in <a href="/fg/annotare/">Annotare</a>. Follow the guides below for information about data formats and what metadata to include. Note that larger experiments with more than 1000 raw data files in one experiment are started in Annotare but will be completed manually with the help of a curator.</p>
  <br>
  <h3 id="SampleInfo">Sample information</h3>
  <h4>Required sample attributes</h4>
  <div class="row">
    <div class="columns medium-11 medium-push-1">
    <p>Minimum sample attributes for primary cells from different species and cell lines</p>
    <table>
      <thead>
        <tr>
          <th>Human <i class="icon icon-species" data-icon="H"></i></th>
          <th>Vertebrates <i class="icon icon-species" data-icon="M"></i></th>
          <th>Plants <i class="icon icon-species" data-icon="P"></i></th>
          <th>Cell lines <i class="icon icon-species" data-icon="Y"></i></th>
          <th>Comment</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>organism</td>
          <td>organism</td>
          <td>organism</td>
          <td>organism</td>
          <td></td>
        </tr>
        <tr>
          <td></td>
          <td>strain or breed</td>
          <td>cultivar or ecotype</td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>age</td>
          <td>age</td>
          <td>age</td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>developmental stage</td>
          <td>developmental stage</td>
          <td>developmental stage</td>
          <td>developmental stage</td>
          <td></td>
        </tr>
        <tr>
          <td>sex</td>
          <td>sex</td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>disease</td>
          <td></td>
          <td></td>
          <td>disease</td>
          <td></td>
        </tr>
        <tr>
          <td></td>
          <td>genotype</td>
          <td>genotype</td>
          <td>genotype</td>
          <td><em>genetic modification</em></td>
        </tr>
        <tr>
          <td>organism part</td>
          <td>organism part</td>
          <td></td>
          <td>organism part</td>
          <td></td>
        </tr>
        <tr>
          <td>cell type*</td>
          <td>cell type*</td>
          <td></td>
          <td>cell type</td>
          <td><em>* if known, see comment below</em></td>
        </tr>
        <tr>
          <td>individual</td>
          <td>individual</td>
          <td></td>
          <td></td>
          <td><em>donor or animal ID</em></td>
        </tr>
        <tr>
          <td></td>
          <td></td>
          <td></td>
          <td>cell line</td>
          <td><em>name of commercial cell line</em></td>
        </tr>
      </tbody>
    </table>
  </div>
  </div>
  <br>
  <h4>Mandatory information that may be applicable</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>
        <h5>Unknown cell type</h5>
        <p> If your experiment involves discovering new cell types or you work with pooled cells from a whole organ or organism, the precise cell type might be undefined at the start of the experiment but inferred later during the analysis. In that case, annotate the samples with the inferred cells type using the attribute <strong>"inferred cell type"</strong>. The attribute "cell type" should refer to the input cell type that is known at the start of the experiment. <a href="/arrayexpress/experiments/E-MTAB-5953/samples/" class="readmore" target="_blank">Example: E-MTAB-5953</a>
        </p>
        <hr />
      </li>

      <li>
        <h5>Experimental variable</h5>
        <p> If the purpose of your experiment is comparing the heterogeneity within one cell population, you might not have any other experimental variables. In this case, you can use the term <strong>"single cell identifier"</strong> as experimental variable and enter the cell IDs as values. <a href="/arrayexpress/experiments/E-MTAB-6142/samples/" class="readmore" target="_blank">Example: E-MTAB-6142</a>
        </p>
        <hr />
      </li>

      <li>
        <h5>Quality control pre-analysis</h5>
        <p> If your single-cell experiment included visual inspection of the cells before lysis, you can include this information in the sample attributes as <strong>"single cell well quality"</strong>. The values for this attribute should be "OK" or "not OK" (i.e. the well should be discarded from data analysis). If you have more detailed information about the "bad" wells, the following terms should be used:</p>
          <ul>
            <li>"debris"</li>
            <li>"multiple cells" or number of cells if known</li>
            <li>"dead cell"</li>
            <li>"no cell"</li>
          </ul>
        <a href="/arrayexpress/experiments/E-MTAB-5522/samples/" class="readmore" target="_blank">Example: E-MTAB-5522</a>

        <hr />
      </li>

      <li>
        <h5>Quality control post-analysis</h5>
        <p> You might have quality information about the dataset, e.g. which cells were excluded from your analysis due to insufficient sequencing quality. You can use the attribute <strong>"post analysis single cell quality"</strong> and annotate each cell with "pass" or "fail".
        <a href="/arrayexpress/experiments/E-MTAB-5522/samples/" class="readmore" target="_blank">Example: E-MTAB-5522</a>
        </p><hr />
      </li>
    </ul>
  </div>
  </div>

  <div class="row">
    <div class="columns medium-11 medium-push-1">
      <p>Examples of additional single-cell sample attributes</p>
      <table>
        <thead>
          <tr>
            <th width="100">Example</th>
            <th width="200">single cell identifier (experimental variable)</th>
            <th width="150">inferred cell type</th>
            <th>single cell well quality</th>
            <th>post-analysis single cell quality</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>Sample 1</td>
            <td>cell 1</td>
            <td>cell type A</td>
            <td>OK</td>
            <td>pass</td>
          </tr>
          <tr>
            <td>Sample 2</td>
            <td>cell 2</td>
            <td>cell type B</td>
            <td>OK</td>
            <td>pass</td>
          </tr>
          <tr>
            <td>Sample 3</td>
            <td>cell 3</td>
            <td>not applicable</td>
            <td>OK</td>
            <td>fail</td>
          </tr>
          <tr>
            <td>Sample 4</td>
            <td>cell 4</td>
            <td>cell type C</td>
            <td>OK</td>
            <td>pass</td>
          </tr>
          <tr>
            <td>Sample 5</td>
            <td>cell 5</td>
            <td>not applicable</td>
            <td>2 cells</td>
            <td>fail</td>
          </tr>
          <tr>
            <td>Sample 6</td>
            <td>cell 6</td>
            <td>not applicable</td>
            <td>debris</td>
            <td>fail</td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
  <br>
  <h4>Sample collection protocol</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>
        <h5>Method of cell singularisation</h5>
        <p> Please include a description of how the cells were treated and separated into single-cells, e.g. FACS or microfluidics (Fluidigm).</p><hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h3 id="LibraryInfo">2. Library and sequencing information</h3>
  <h4>Library construction protocol</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>
        <h5>Single-cell library construction</h5>
        <p>Please mention the type of single-cell library method that was used (e.g. Smart-seq2, 10x, Drop-seq) and give any relevant literature references.</p><hr />
      </li>

      <li>
        <h5>Library construction kit</h5>
        <p>Please include the name, manufacturer and catalogue number of the library preparation kit(s) that were used.</p><hr />
      </li>
    </ul>
  </div>
  </div>

  <br>
  <h4>Sequencing protocol</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>
        <h5>Technical replicates</h5>
        <p>Please include details about technical replicates, e.g. if the same libraries were sequenced multiple times or across several lanes. </p><hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h4>Spike-in RNAs</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>
        <h5>Commercial spike-ins</h5>
        <p>If you have used spike-in that are commercially available, please include the kit name, catalogue number and dilution in the library construction protocol.</p>
      </li>
      <hr />
      <li>
        <h5>Custom spike-ins</h5>
        <p>If you have used a non-commercial spike-in set, describe this in the library construction protocol. Additionally, upload a table (tab-delimited text) with the names and concentrations of the spike-ins, and a <em>fasta</em> file containing the nucleotide sequences. <a href="/arrayexpress/files/E-MTAB-3624/E-MTAB-3624.additional.1.zip" class="readmore">Example: E-MTAB-3624.additional.1.zip</a>
        </p>
      </li>
      <hr />
      <li>
        <h5>Spike-in annotation</h5>
        <p>You can include spike-in information at the cell level in the sample annotation under <strong>"spike in"</strong> and <strong>"spike in dilution"</strong>.</p>
        <p>Example of spike-in annotation</p>
        <table>
          <thead>
            <tr>
              <th>Example</th>
              <th>spike in</th>
              <th>spike in dilution</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>Sample 1</td>
              <td>ERCC mix1</td>
              <td>1:40000</td>
            </tr>
            <tr>
              <td>Sample 2</td>
              <td>ERCC mix2</td>
              <td>1:40000</td>
            </tr>
            <tr>
              <td>Sample 3</td>
              <td>ERCC mix1 and mix2</td>
              <td>1:40000</td>
            </tr>
          </tbody>
        </table>
        <hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h4>Unique molecular identifiers</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>
        <p>If your single-cell protocol uses unique molecular identifiers (UMIs), all relevant information about the UMI barcodes should be included to enable re-analysis of the data. We suggest to include the following:</p>
        <ul>
          <li><strong>UMI barcode read</strong> (the file that contains the UMI). Values: read1/read2/index1/index2</li>
          <li><strong>UMI barcode offset</strong> (start position of UMI barcode in the sequence). Values: (number, 0 for start of read)</li>
          <li><strong>UMI barcode size</strong> (length of UMI barcode in bp). Values: (number)</li>
        </ul>
        <hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h3 id="FileFormats">File formats</h3>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>
        <h5>Raw data</h5>
        <p>Raw read data should be submitted in <em>fastq.gz</em> format. Prepare one file per cell (or 2 if you have used paired-end sequencing), following the <a href="http://ena-docs.readthedocs.io/en/latest/format_01.html">general recommendations from the European Nucleotide Archive (ENA)</a>. An exception are droplet-based technologies like 10x and Drop-seq; for those protocols please follow the <a href="#Droplet">guide below</a>.</p>
      <hr />
      </li>
      <li>
        <h5>Processed data</h5>
        <p>Processed data is welcome in addition to raw data. Most commonly this would be a raw or normalised read count matrix. The format of any matrix file should be tab-delimited text. But also other analysis results files can be included, e.g. alignment files or annotation tracks.</p>
        <hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h3 id="Droplet">Droplet-based technologies</h3>

  <p>For droplet-based technologies, like 10x and Drop-seq, we allow submission of multiplexed data. A few extra rules apply here.</p>

  <h4>Sample annotation</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li>Create and annotate <strong>1 sample per library</strong> (often containing several thousands of individual cells), instead of 1 sample per cell.
        <hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h4>Library layout</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li><p>For 10x technology, include the <strong>version of the 10x chemistry</strong>, e.g. v1 or v2, and provide the exact name of the library construction kit and catalogue number in the "library construction protocol".
      </p><hr />
      </li>
      <li><p>For <strong>other large-scale single-cell sequencing methods</strong> or where modifications were done to the standard 10x protocol, please include specifications about the multiplexing and barcodes. We suggest to include the following to specify the positions and size of the barcodes:</p>
        <table>
          <thead>
            <tr>
              <th>Attribute</th>
              <th>Description</th>
              <th>Possible values</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>cDNA read</td>
              <td>the file that contains the cDNA read</td>
              <td>index1/index2/read1/read2</td>
            </tr>
            <tr>
              <td>cDNA read offset</td>
              <td>offset in sequence for cDNA read (in bp)</td>
              <td>(number, 0 for start of read)</td>
            </tr>
            <tr>
              <td>cDNA read size</td>
              <td>length of cDNA read (in bp)</td>
              <td>(number)</td>
            </tr>
            <tr>
              <td>UMI barcode read</td>
              <td>the file that contains the UMI barcode read</td>
              <td>index1/index2/read1/read2</td>
            </tr>
            <tr>
              <td>UMI barcode offset</td>
              <td>offset in sequence for UMI barcode read (in bp)</td>
              <td>(number, 0 for start of read)</td>
            </tr>
            <tr>
              <td>UMI barcode size</td>
              <td>length of UMI barcode read (in bp)</td>
              <td>(number)</td>
            </tr>
            <tr>
              <td>cell barcode read</td>
              <td>the file that contains the cell barcode read</td>
              <td>index1/index2/read1/read2</td>
            </tr>
            <tr>
              <td>cell barcode offset</td>
              <td>offset in sequence for cell barcode read (in bp)</td>
              <td>(number, 0 for start of read)</td>
            </tr>
            <tr>
              <td>cell barcode size</td>
              <td>length of cell barcode read (in bp)</td>
              <td>(number)</td>
            </tr>
            <tr>
              <td>sample barcode read</td>
              <td>the file that contains the sample barcode read</td>
              <td>index1/index2/read1/read2</td>
            </tr>
            <tr>
              <td>sample barcode offset</td>
              <td>offset in sequence for sample barcode read (in bp)</td>
              <td>(number, 0 for start of read)</td>
            </tr>
            <tr>
              <td>sample barcode size</td>
              <td>length of sample barcode read (in bp)</td>
              <td>(number)</td>
            </tr>
          </tbody>
        </table>
        <hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h4>File format for droplet-based technologies</h4>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li><p>For <strong>10x technology</strong>, please provide <em>fastq.gz</em> files as <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq">generated by CellRanger software from bcl files</a>. These are usually 3-4 <em>fastq.gz</em> files per library containing the cDNA read and several barcode reads in known positions.</p>
        <p>We can currently validate files that follow the 10x file naming conventions:</p>
        <table>
          <thead>
            <tr>
              <th>10x version 1</th>
              <th>10x version 2</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>R1 + R2 + R3 + I1 or RA + I1 + I2</td>
              <td>R1 + R2 + I1</td>
            </tr>
          </tbody>
        </table>
           <hr />
      </li>
      <li><p><strong>Processed data</strong>: Since there is no information about the individual cells at the sample annotation or file level, it makes sense to include processed data, e.g. a read count matrix, to describe the analysis results like the "inferred cell types" (see <a href="#SampleInfo">above</a>).
        </p> <hr />
      </li>
      <li><p><strong>Additional files</strong>: It is encouraged to upload any additional files that facilitate data analysis, e.g. text files containing lists of known barcodes in the library. These files will be linked to the experiment and don't need to be associated with any specific sample in the submission. Please leave a description of such files in an appropriate protocol, e.g. the normalisation data transformation protocol. <a href="https://www.ebi.ac.uk/arrayexpress/files/E-MTAB-6153/E-MTAB-6153.additional.1.zip/E-MTAB-6153_sample_barcodes.txt" class="readmore" target="_blank">Example: E-MTAB-6153_sample_barcodes.txt</a>
        </p><hr />
      </li>
    </ul>
  </div>
  </div>
  <br>
  <h3 id="Tips">A few more tips</h3>
  <div class="row">
  <div class="columns medium-11 medium-push-1">
    <ul class="no-bullet">
      <li><p>Use experiment type "RNA-seq of coding RNA from single cells" for single-cell RNA-sequencing experiments.</p><hr /></li>
      <li><p>Make sure to check the library strand information of your library protocol or kit and enter the relevant strand (see <a href="/fg/annotare/help/seq_lib_spec.html">guide to sequencing library information</a>).</p><hr /></li>
      <li><p>If an attribute or parameter in a protocol is different for different files, please include this information at the level of sample annotation. Alternatively, create multiple protocols of the same type and assign them to individual samples (e.g. if you have included files from different sequencer models, create two sequencing protocols).</p><hr /></li>
    </ul>
  </div>
  </div>
  </div>

  <!-- side navigation -->
  <div class="small-12 medium-3 columns sidebar">
    <ul class="vertical menu accordion-menu" data-accordion-menu>
      <li><a href="/fg/annotare/help/" class="active" >Submission guide</a></li>
      <li><a>Before you start</a>
        <ul class="menu vertical nested">
          <li><a href="what_to_submit.html">Submission overview</a></li>
          <li><a href="">Accepted file formats</a>
            <ul>
              <li><a href="accepted_raw_ma_file_formats.html">Microarray raw data</a></li>
              <li><a href="accepted_processed_ma_file_formats.html">Microarray processed data</a></li>
              <li><a href="UHTS_submissions.html#HowToSubmit">Sequencing raw data</a></li>
              <li><a href="UHTS_submissions.html#processed_data">Sequencing processed data</a></li>
            </ul>
          </li>
          <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
          <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
        </ul>
      </li>
      <li><a>Annotare step-by-step</a>
        <ul class="menu vertical nested is-active">
          <li><a href="getting_started.html">Getting started</a></li>
          <li><a href="time_saving_features.html">Time-saving features</a></li>
          <li><a href="describe_exp.html">Describe experiment</a></li>
          <li><a href="sample_attributes.html">Sample attributes and protocols</a></li>
          <li><a href="">Technology-specific details</a>
            <ul class="menu vertical nested is-active">
              <li><a href="seq_lib_spec.html">Sequencing library information</a></li>
              <li><a class="active" href="singlecell_submission_guide.html">Single cell information</a></li>
              <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
          </li>
          <li><a href="">Upload and assign files</a>
            <ul>
              <li><a href="file_upload.html#direct">Direct upload</a></li>
              <li><a href="ftp_upload.html">FTP/Aspera upload</a></li>
              <li><a href="file_upload.html#ftp">Import FTP/Aspera uploaded files</a></li>
              <li><a href="assign_files.html">Assign files</a></li>
            </ul>
          </li>
          <li><a href="">Validate and submit</a>
            <ul>
              <li><a href="validate_exp.html#validate">What is validation?</a></li>
              <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
              <li><a href="submit_exp.html#save_sub">Saving a copy of your submission</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="">After submission</a>
        <ul class="menu vertical nested ">
          <li><a href="after_sub_private_access.html">Access private experiments</a></li>
          <li><a href="after_sub_modify_data.html">Modify or remove an existing experiment</a></li>
          <li><a href="after_sub_data_availability.html">Data availability and release policy</a></li>
        </ul>
      </li>
    </ul>
  </div> <!-- /medium-3 -->

</section>
