<!doctype html>
<html lang="en">
<script>
  document.title = "Single-cell submission guide < Submission Guide < Annotare < EMBL-EBI";
</script>
<body>

<style>
  table {
    font-size: 14px;
  }
</style>

<section id="annotare-help">
  <div class="small-12 medium-9 columns">

  <h2>Single-cell submission guide</h2>
  <br>
  <ol>
    <li><a href="#SampleInfo">Sample information</a></li>
    <li><a href="#LibraryInfo">Single cell library information</a></li>
    <li><a href="#FileFormats">File types and formats</a></li>
    <li><a href="#Protocols">Protocols</a></li>
  </ol>
  <br>

  <p>Start your single-cell submission using the single cell sequencing template in <a href="https://www.ebi.ac.uk/fg/annotare/">Annotare</a>. Follow the guides below for information about data formats and what metadata to include. Note that larger experiments with more than 1000 raw data files in one experiment are started in Annotare but will be completed manually with the help of a curator.</p>
  <br>
  <h3 id="SampleInfo">Sample information</h3>

  <h4>What is a "sample" in a single-cell experiment?</h4>
  <p>The general rule is to create and annotate <strong>1 sample per library</strong>. In plate-based methods, like Smart-seq2, one library will correspond to a single cell, which will be the sample. In case you have multiplexed data from droplet-based platforms like 10x, one library will contain several thousands of individual cells. The following attributes are suggested for single cell experiments and can be removed if they don't apply. </p>

  <h4>Single cell level attributes</h4>

    <ul class="no-bullet">
      <li>
        <p><strong>Inferred cell type</strong><br>
        If your experiment involves discovering new cell types or you work with pooled cells from a whole organ or organism, the precise cell type might be undefined at the start of the experiment but inferred later during the analysis. In that case, annotate the samples with the inferred cells type using the attribute <strong>"inferred cell type"</strong>. The attribute "cell type" should refer to the input cell type that is known at the start of the experiment. <br>
        If you have multiplexed data and cannot enter this in the samples table, please include the annotations in a separate processed data file (<a href="#ProcessedData">see below</a>).<br>
        <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5953/samples/" class="readmore" target="_blank">Example: E-MTAB-5953</a>
        </p>
      </li>

      <li>
        <p><strong>Single cell identifier (experimental variable)</strong><br>
        If the purpose of your experiment is comparing the heterogeneity within one cell population, you might not have any other experimental variables. In this case, you can use the term <strong>"single cell identifier"</strong> as experimental variable and enter the cell IDs as values. <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6142/samples/" class="readmore" target="_blank">Example: E-MTAB-6142</a>
        </p>
      </li>

      <li>
        <p><strong>Single cell well quality (pre-analysis)</strong><br>
        If your single-cell experiment included visual inspection of the cells before lysis, you can include this information in the sample attributes as <strong>"single cell well quality"</strong>. The values for this attribute should be "OK" or "not OK" (i.e. the well should be discarded from data analysis). If you have more detailed information about the "bad" wells, the following terms should be used:</p>
          <ul>
            <li>"debris"</li>
            <li>"multiple cells" or number of cells if known</li>
            <li>"dead cell"</li>
            <li>"no cell"</li>
          </ul>
        <p><a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5522/samples/" class="readmore" target="_blank">Example: E-MTAB-5522</a></p>
      </li>

      <li>
        <p><strong>Post-analysis well quality</strong><br>
        You might have quality information about the dataset, e.g. which cells were excluded from your analysis due to insufficient sequencing quality. You can use the attribute <strong>"post analysis single cell quality"</strong> and annotate each cell with "pass" or "fail".
        <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5522/samples/" class="readmore" target="_blank">Example: E-MTAB-5522</a>
        </p>
      </li>

    </ul>

  <br>
  <h5>Examples of single-cell sample attributes</h5>

  <table>
    <thead>
      <tr>
        <th width="100">Example</th>
        <th width="200">single cell identifier (experimental variable)</th>
        <th width="150">inferred cell type</th>
        <th>single cell well quality</th>
        <th>post-analysis single cell quality</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>Sample 1</td>
        <td>cell 1</td>
        <td>cell type A</td>
        <td>OK</td>
        <td>pass</td>
      </tr>
      <tr>
        <td>Sample 2</td>
        <td>cell 2</td>
        <td>cell type B</td>
        <td>OK</td>
        <td>pass</td>
      </tr>
      <tr>
        <td>Sample 3</td>
        <td>cell 3</td>
        <td>not applicable</td>
        <td>OK</td>
        <td>fail</td>
      </tr>
      <tr>
        <td>Sample 4</td>
        <td>cell 4</td>
        <td>cell type C</td>
        <td>OK</td>
        <td>pass</td>
      </tr>
      <tr>
        <td>Sample 5</td>
        <td>cell 5</td>
        <td>not applicable</td>
        <td>2 cells</td>
        <td>fail</td>
      </tr>
      <tr>
        <td>Sample 6</td>
        <td>cell 6</td>
        <td>not applicable</td>
        <td>debris</td>
        <td>fail</td>
      </tr>
    </tbody>
  </table>

  <hr />
  <br>
  <h3 id="LibraryInfo">Single cell library information</h3>

  <h4>1. Pick library construction method </h4>
  <p>Select the type of single cell <strong>library construction</strong> that you followed from <a href="https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0010183&viewMode=All&siblings=false">controlled vocabulary</a> drop-down menu. For some of the most commonly used methods, this will pre-fill the other fields with the default values. These can still be adjusted afterwards. </p>

  <h4>2. Refine parameters</h4>
  <p>Check each parameter carefully, selecting the suitable option from the drop-down menu. For some commonly used library constructions methods Annotare will show the default parameters of the protocol, you may still need to adjust those.</p>

  <br>

    <h5>General description</h5>
    <ul class="no-bullet">
      <li>
        <p><strong>Single Cell Isolation</strong><br>
        The method used for selection of single cells, to allow individual cell barcoding</p>
      </li>
      <li>
        <p><strong>End Bias</strong><br>
        The end of the nucleic acid molecule that is preferentially sequenced (bias in read distribution)</p>
      </li>
      <li>
        <p><strong>Input molecule</strong><br>
        The type or fraction of nucleic acid that was captured in the library</p>
      </li>
      <li>
        <p><strong>Primer</strong><br>
        The type of primer used for reverse-transcription</p>
      </li>
    </ul>

    <h5>Spike-in RNAs</h5>
    <ul class="no-bullet">
      <li>
        <p><strong>Commercial spike-ins</strong><br>
        If you have used spike-in that are commercially available, please include the kit name, catalogue number and dilution in the library construction protocol.</p>
      </li>
      <li>
        <p><strong>Custom spike-ins</strong><br>
        If you have used a non-commercial spike-in set, describe this in the library construction protocol. Additionally, upload a table (tab-delimited text) with the names and concentrations of the spike-ins, and a <em>fasta</em> file containing the nucleotide sequences. <a href="https://www.ebi.ac.uk/arrayexpress/files/E-MTAB-3624/E-MTAB-3624.additional.1.zip" class="readmore">Example: E-MTAB-3624.additional.1.zip</a>
        </p>
      </li>
      <li>
        <p><strong>Spike-in annotation</strong><br>
        You can include spike-in information at the cell level in the sample annotation under <strong>"spike in"</strong> and <strong>"spike in dilution"</strong>.</p>
        <p>Example of spike-in annotation</p>
        <table>
          <thead>
            <tr>
              <th>Example</th>
              <th>spike in</th>
              <th>spike in dilution</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>Sample 1</td>
              <td>ERCC mix1</td>
              <td>1:40000</td>
            </tr>
            <tr>
              <td>Sample 2</td>
              <td>ERCC mix2</td>
              <td>1:40000</td>
            </tr>
            <tr>
              <td>Sample 3</td>
              <td>ERCC mix1 and mix2</td>
              <td>1:40000</td>
            </tr>
          </tbody>
        </table>
      </li>
    </ul>

    <h5>Unique Molecular Identifiers</h5>
    <ul class="no-bullet">
      <li>
        <p>If your single-cell protocol uses unique molecular identifiers (UMIs), all relevant information about the UMI barcodes should be included to enable re-analysis of the data. We suggest to include the following:</p>
        <ul>
          <li><strong>UMI barcode read</strong> (the file that contains the UMI). Values: read1/read2/index1/index2</li>
          <li><strong>UMI barcode offset</strong> (start position of UMI barcode in the sequence). Values: (number, 0 for start of read)</li>
          <li><strong>UMI barcode size</strong> (length of UMI barcode in bp). Values: (number)</li>
        </ul>
      </li>
    </ul>


  <br>
  <h4>3. Define library layout for multiplexed data</h4>

  <p>For 10x technology, ensure you have selected the correct <strong>version of the 10x chemistry</strong>, e.g. v2 or v3. <em>Note that Annotare currently only provides templates for gene expression experiments</em>.</p>
  <p>For <strong>other large-scale single-cell sequencing methods</strong> or where modifications were done to the standard 10x protocol, please include specifications about the multiplexing and barcodes. We suggest to include the following to specify the positions and size of the barcodes:</p>

  <table>
        <thead>
          <tr>
            <th>Attribute</th>
            <th>Description</th>
            <th>Possible values</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>cDNA read</td>
            <td>the file that contains the cDNA read</td>
            <td>index1/index2/read1/read2</td>
          </tr>
          <tr>
            <td>cDNA read offset</td>
            <td>offset in sequence for cDNA read (in bp)</td>
            <td>(number, 0 for start of read)</td>
          </tr>
          <tr>
            <td>cDNA read size</td>
            <td>length of cDNA read (in bp)</td>
            <td>(number)</td>
          </tr>
          <tr>
            <td>UMI barcode read</td>
            <td>the file that contains the UMI barcode read</td>
            <td>index1/index2/read1/read2</td>
          </tr>
          <tr>
            <td>UMI barcode offset</td>
            <td>offset in sequence for UMI barcode read (in bp)</td>
            <td>(number, 0 for start of read)</td>
          </tr>
          <tr>
            <td>UMI barcode size</td>
            <td>length of UMI barcode read (in bp)</td>
            <td>(number)</td>
          </tr>
          <tr>
            <td>cell barcode read</td>
            <td>the file that contains the cell barcode read</td>
            <td>index1/index2/read1/read2</td>
          </tr>
          <tr>
            <td>cell barcode offset</td>
            <td>offset in sequence for cell barcode read (in bp)</td>
            <td>(number, 0 for start of read)</td>
          </tr>
          <tr>
            <td>cell barcode size</td>
            <td>length of cell barcode read (in bp)</td>
            <td>(number)</td>
          </tr>
          <tr>
            <td>sample barcode read</td>
            <td>the file that contains the sample barcode read</td>
            <td>index1/index2/read1/read2</td>
          </tr>
          <tr>
            <td>sample barcode offset</td>
            <td>offset in sequence for sample barcode read (in bp)</td>
            <td>(number, 0 for start of read)</td>
          </tr>
          <tr>
            <td>sample barcode size</td>
            <td>length of sample barcode read (in bp)</td>
            <td>(number)</td>
          </tr>
        </tbody>
      </table>

  <hr />

  <br>
  <h3 id="FileFormats">File types and formats</h3>

    <h4 id="RawData">Raw data</h4>
    <p>In addition to the rules below, all raw data must conform to the general <a href="accepted_sequencing_file_formats.html">accepted sequencing file formats</a>.</p>

    <p>Raw read data should be submitted in <em>fastq.gz</em> format. Prepare one file per cell (or 2 if you have used paired-end sequencing), following the <a href="http://ena-docs.readthedocs.io/en/latest/format_01.html">general recommendations from the European Nucleotide Archive (ENA)</a>. An exception are droplet-based technologies like 10x and Drop-seq and other methods for which data is not demultiplexed, where we allow submission of multiplexed data. A few extra rules apply here.</p>

    <p>For <strong>10x technology</strong>, please provide <em>fastq.gz</em> files, for example as <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq">generated by CellRanger software from bcl files</a>. These are usually 2-4 <em>fastq.gz</em> files per library, containing the cDNA read and several barcode reads in known positions.</p>
      <p>We can currently validate files that follow the 10x file naming conventions:</p>
      <ul>
        <li>R1 + R2 (+ I1)</li>
        <li>R1 + R2 + R3 + I1</li>
        <li>RA + I1 + I2</li>
      </ul>

    <p>For <strong>other formats</strong> where the data cannot be represented as 2 paired-end fastq files, please convert your raw data to <strong>unaligned BAM format</strong>. We recommend using the following SAM specification tags for the typical single cell barcode sequences: </p>
    <ul>
      <li> BC: Sample barcode sequence</li>
      <li> QT: Sample barcode quality</li>
      <li> CR: Cell barcode sequence</li>
      <li> CY: Cell barcode quality</li>
      <li> RX: UMI barcode sequence</li>
      <li> QX: UMI barcode quality</li>
    </ul>

    <hr />
    <br>
    <h4 id="ProcessedData">Processed data</h4>
    <p>Processed data is welcome in addition to raw data. Most commonly this would be a raw or normalised read count matrix. The format of any matrix file should be tab-delimited text. We also accept other commonly generated formats such as sparse matrix files. Also other types of analysis result files can be included, e.g. alignment files, cluster or cell type annotations.</p>

    <p><strong>Inferred cell type annotations</strong> (<a href="#SampleInfo">see above</a>) are a very important piece of metadata that greatly enhance the value of single-cell sequencing data stored in ArrayExpress and we very much encourage all submitters to add this information to their submission. Since information about individual cells can't be entered at the sample annotation or file level for multiplexed data, please include inferred cell type annotations and any other cell-level annotations as a processed data file. Any such file should at least contain the following information, so the correct connection between the annotations and the the relevant samples and sequencing reads can be made:
    </p>

    <table>
      <tbody>
        <tr>
            <td>
              Column header
            </td>
            <td>
              Library ID/Sample ID
            </td>
            <td>
              Cell Barcode
            </td>
            <td>
                Inferred cell type
            </td>
        </tr>
        <tr>
            <td>
              Description:
            </td>
            <td>
                This should match with the raw data file prefix
            </td>
            <td>
                The sequence of the cell barcode
            </td>
            <td>
                assigned cell type annotation
            </td>
        </tr>
        <tr>
            <td>
                <em>Example:</em>
            </td>
            <td>
                <em>Sample1_run1</em>
            </td>
            <td>
                <em>ATGGTCATCGT</em>
            </td>
            <td>
                <em>macrophage</em>
            </td>
        </tr>
      </tbody>
    </table>

    <p><a href="https://www.ebi.ac.uk/arrayexpress/files/E-MTAB-9684/E-MTAB-9684.processed.2.zip/Annotation_FUGI_R_D7119553.txt" class="readmore" target="_blank">Example: E-MTAB-9684</a></p>
    <hr />
    <br>
    <h4 id="AdditionalData">Additional files</h4>
    <p>It is encouraged to upload any additional files that facilitate data analysis, e.g. text files containing lists of known barcodes in the library. These files will be linked to the experiment and don't need to be associated with any specific sample in the submission. Please leave a description of such files in an appropriate protocol, e.g. the normalisation data transformation protocol. <a href="https://www.ebi.ac.uk/arrayexpress/files/E-MTAB-6153/E-MTAB-6153.additional.1.zip/E-MTAB-6153_sample_barcodes.txt" class="readmore" target="_blank">Example: E-MTAB-6153_sample_barcodes.txt</a>
    </p>
    <p>Cell hashing and cell surface antibody readout are other areas where additional files are useful, as there is no standard format for encoding this in Annotare (yet). For <strong>cell hashing</strong> an additional table is needed with the individual sample annotations and respective barcodes. For <strong>cell surface antibody</strong> readout, such as CITE-seq, the file should include the antibody targets and their barcode sequence.
    <a href="https://www.ebi.ac.uk/arrayexpress/files/E-MTAB-9295/E-MTAB-9295.additional.1.zip/E-MTAB-9295_features.txt" class="readmore" target="_blank">Example: E-MTAB-9295_features.txt</a></p>
    <p><em>Note</em>: Currently there is no dedicated way to assign "additional files" in Annotare. Please use the "Processed Data" columns in the file assignment to attach such files to your submission.</p>
    <hr />

  <br>

  <h3 id="Protocols">Protocols</h3>
  <h5>Sample collection protocol</h5>

    <ul class="no-bullet">
      <li>
        <strong>Method of cell singularisation</strong>
        <p> Please include a description of how the cells were treated and separated into single-cells, e.g. FACS or microfluidics (Fluidigm).</p>
      </li>
    </ul>


  <h5>Library construction protocol</h5>

    <ul class="no-bullet">
      <li>
        <p><strong>Single-cell library construction</strong><br>
        Please mention the type of single-cell library method that was used (e.g. Smart-seq2, 10x, Drop-seq) and give any relevant literature references.</p>
      </li>

      <li>
        <p><strong>Library construction kit</strong><br>
        Please include the name, manufacturer and catalogue number of the library preparation kit(s) that were used.</p>
      </li>
    </ul>


  <h5>Sequencing protocol</h5>

    <ul class="no-bullet">
      <li>
        <strong>Technical replicates</strong>
        <p>Please include details about technical replicates, e.g. if the same libraries were sequenced multiple times or across several lanes. </p>
      </li>
    </ul>

  <br>

</div>

  <!-- side navigation -->
  <div class="small-12 medium-3 columns sidebar">
    <ul class="vertical menu accordion-menu" data-accordion-menu>
      <li><a href="index.html" class="active" >Submission guide</a></li>
      <li><a>Before you start</a>
        <ul class="menu vertical nested">
          <li><a href="what_to_submit.html">Submission overview</a></li>
          <li><a href="">Accepted file formats</a>
            <ul>
              <li><a href="accepted_raw_ma_file_formats.html">Microarray raw data</a></li>
              <li><a href="accepted_processed_ma_file_formats.html">Microarray processed data</a></li>
              <li><a href="accepted_sequencing_file_formats.html#raw_data_files">Sequencing raw data</a></li>
              <li><a href="accepted_sequencing_file_formats.html#processed_data">Sequencing processed data</a></li>
            </ul>
          </li>
          <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
          <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
        </ul>
      </li>
      <li><a>Annotare step-by-step</a>
        <ul class="menu vertical nested is-active">
          <li><a href="getting_started.html">Getting started</a></li>
          <li><a href="time_saving_features.html">Time-saving features</a></li>
          <li><a href="describe_exp.html">Describe experiment</a></li>
          <li><a href="sample_attributes.html">Sample attributes and protocols</a></li>
          <li><a href="">Technology-specific details</a>
            <ul class="menu vertical nested is-active">
              <li><a href="seq_lib_spec.html">Sequencing library information</a></li>
              <li><a class="active" href="single-cell_submission_guide.html">Single cell information</a></li>
              <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
          </li>
          <li><a href="">Upload and assign files</a>
            <ul>
              <li><a href="file_upload.html#direct">Direct upload</a></li>
              <li><a href="ftp_upload.html">FTP/Aspera upload</a></li>
              <li><a href="file_upload.html#ftp">Import FTP/Aspera uploaded files</a></li>
              <li><a href="assign_files.html">Assign files</a></li>
            </ul>
          </li>
          <li><a href="">Validate and submit</a>
            <ul>
              <li><a href="validate_exp.html#validate">What is validation?</a></li>
              <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
              <li><a href="submit_exp.html#save_sub">Saving a copy of your submission</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="">After submission</a>
        <ul class="menu vertical nested ">
          <li><a href="after_sub_private_access.html">Access private experiments</a></li>
          <li><a href="after_sub_modify_data.html">Modify or remove an existing experiment</a></li>
          <li><a href="after_sub_data_availability.html">Data availability and release policy</a></li>
        </ul>
      </li>
    </ul>
  </div> <!-- /medium-3 -->

</section>

</body>
</html>
