<section id="annotare-help">
  <div class="small-12 medium-9 columns">

    <h2>Submission overview</<h2>
    <h3>What data can be submitted?</h3>
    <div>
      <p>ArrayExpress accepts all <strong>functional genomics data</strong> generated from <strong>microarray</strong> or <strong>next-generation sequencing (NGS)</strong> platforms. Popular experiment types are transcription profiling (mRNA and miRNA), SNP genotyping, chromatin immunoprecipitation (ChIP) and comparative genomic hybridisation. Here is the <a href="#experiment_types">full list of experiment types in ArrayExpress</a>.</p>
      <p class="ngs">We currently <strong>don't accept</strong> metagenomics/metatranscriptomics data (please submit to the <a href="/metagenomics/submission">EBI Metagenomics service</a>) or de novo transcriptome assembly data (the raw RNA-seq reads should be submitted to ArrayExpress, but the assembled transcriptome file directly to the <a href="/ena/submit">European Nucleotide Archive</a>).</p>
    </div>

    <h3 id="experiment_types">Experiment types in ArrayExpress</h3>

    <div class="accordion"><p>Click to show list of experiment types that can be deposited in ArrayExpress</p></div>
    <div class="panel">
      <table class="table">
        <thead>
          <tr>
              <th>Experiment Type</th><th>Definition (EFO term accession)</th><th>Example</th>
          </tr>
        </thead>
        <tbody>
          <tr>
              <td>4C</td>
              <td><a href="/efo/EFO_0007690">EFO_0007690</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2180">E-MTAB-2180</a></td>
          </tr>
          <tr>
              <td>antigen profiling</td>
              <td><a href="/efo/EFO_0000747">EFO_0000747</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3606/">E-MTAB-3606</a></td>
          </tr>
          <tr>
              <td>ATAC-seq</td>
              <td><a href="/efo/EFO_0007045">EFO_0007045</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3972">E-MTAB-3972</a></td>
          </tr>
          <tr>
              <td>Bisulfite-seq</td>
              <td><a href="/efo/EFO_0003753">EFO_0003753</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-1042/">E-MTAB-1042</a></td>
          </tr>
          <tr>
              <td>Capture-C</td>
              <td><a href="/efo/EFO_0007691">EFO_0007691</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-4845">E-MTAB-4845</a></td>
          </tr>
          <tr>
              <td>ChIP-chip by array</td>
              <td><a href="/efo/EFO_0002760">EFO_0002760</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2804">E-MTAB-2804</a></td>
          </tr>
          <tr>
              <td>ChIP-chip by SNP array</td>
              <td><a href="/efo/EFO_0002764">EFO_0002764</a></td>
              <td><a href="/arrayexpress/experiments/E-GEOD-22306/">E-GEOD-22306</a></td>
          </tr>
          <tr>
              <td>ChIP-chip by tiling array</td>
              <td><a href="/efo/EFO_0002762">EFO_0002762</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-1402/">E-MTAB-1402</a></td>
          </tr>
          <tr>
              <td>ChIP-seq</td>
              <td><a href="/efo/EFO_0002692">EFO_0002692</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3631/">E-MTAB-3631</a></td>
          </tr>
          <tr>
              <td>CLIP-seq</td>
              <td><a href="/efo/EFO_0003143">EFO_0003143</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-1371/">E-MTAB-1371</a></td>
          </tr>
          <tr>
              <td>comparative genomic hybridization by array</td>
              <td><a href="/efo/EFO_0000749">EFO_0000749</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2293/">E-MTAB-2293</a></td>
          </tr>
          <tr>
              <td>DNA-seq</td>
              <td><a href="/efo/EFO_0002693">EFO_0002693</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3109/">E-MTAB-3109</a></td>
          </tr>
          <tr>
              <td>FAIRE-seq</td><td><a href="/efo/EFO_0004428">EFO_0004428</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3199">E-MTAB-3199</a></td>
          </tr>
          <tr>
              <td>genotyping by array</td><td><a href="/efo/EFO_0002767">EFO_0002767</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3729">E-MTAB-3729</a></td>
          </tr>
          <tr>
              <td>genotyping by high throughput sequencing</td>
              <td><a href="/efo/EFO_0002771">EFO_0002771</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-1794/">E-MTAB-1794</a></td>
          </tr>
          <tr>
              <td>GRO-seq</td>
              <td><a href="/efo/EFO_0005227">EFO_0005227</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3626">E-MTAB-3626</a></td>
          </tr>
          <tr>
              <td>Hi-C</td>
              <td><a href="/efo/EFO_0007693">EFO_0007693</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2414">E-MTAB-2414</a></td>
          </tr>
          <tr>
              <td>MeDIP-seq</td>
              <td><a href="/efo/EFO_0003749">EFO_0003749</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3051/">E-MTAB-3051</a></td>
          </tr>
          <tr>
              <td>metabolomic profiling</td>
              <td><a href="/efo/EFO_0000752">EFO_0000752</a></td>
              <td><a href="/arrayexpress/experiments/E-TABM-290/">E-TABM-290</a></td>
          </tr>
          <tr>
              <td>methylation profiling by array</td>
              <td><a href="/efo/EFO_0002759">EFO_0002759</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3709">E-MTAB-3709</a></td>
          </tr>
          <tr>
              <td>methylation profiling by high throughput sequencing</td>
              <td><a href="/efo/EFO_0002761">EFO_0002761</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3181/">E-MTAB-3181</a></td>
          </tr>
          <tr>
              <td>microRNA profiling by array</td>
              <td><a href="/efo/EFO_0000753">EFO_0000753</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3303/">E-MTAB-3303</a></td>
          </tr>
          <tr>
              <td>microRNA profiling by high throughput sequencing</td>
              <td><a href="/efo/EFO_0002896">EFO_0002896</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3251/">E-MTAB-3251</a></td>
          </tr>
          <tr>
              <td>MNase-seq</td>
              <td><a href="/efo/EFO_0003751">EFO_0003751</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2671/">E-MTAB-2671</a></td>
          </tr>
          <tr>
              <td>proteomic profiling by array</td>
              <td><a href="/efo/EFO_0002765">EFO_0002765</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-1245/">E-MTAB-1245</a></td>
          </tr>
          <tr>
              <td>proteomic profiling by mass spectrometer</td>
              <td><a href="/efo/EFO_0002766">EFO_0002766</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-869">E-MTAB-869</a></td>
          </tr>
          <tr>
              <td>RIP-chip by array</td>
              <td><a href="/efo/EFO_0005517">EFO_0005517</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3881/">E-MTAB-3881</a></td>
          </tr>
          <tr>
              <td>Ribo-seq</td>
              <td><a href="/efo/EFO_0008891">EFO_0008891</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-8039/">E-MTAB-8039</a></td>
          </tr>
          <tr>
              <td>RIP-seq</td>
              <td><a href="/efo/EFO_0005310">EFO_0005310</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3195/">E-MTAB-3195</a></td>
          </tr>
          <tr>
              <td>RNA-seq of coding RNA</td>
              <td><a href="/efo/EFO_0003738">EFO_0003738</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-1147/">E-MTAB-1147</a></td>
          </tr>
          <tr>
              <td>RNA-seq of coding RNA from single cells</td>
              <td><a href="/efo/EFO_0005684">EFO_0005684</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2805/">E-MTAB-2805</a></td>
          </tr>
          <tr>
              <td>RNA-seq of non coding RNA</td>
              <td><a href="/efo/EFO_0003737">EFO_0003737</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2206/">E-MTAB-2206</a></td>
          </tr>
          <tr>
              <td>RNA-seq of non coding RNA from single cells</td>
              <td><a href="/efo/EFO_0005685">EFO_0005685</a></td>
              <td></td>
          </tr>
          <tr>
              <td>RNA-seq of total RNA</td>
              <td><a href="/efo/EFO_0009653">EFO_0009653</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-8118">E-MTAB-8118</a></td>
          </tr>
          <tr>
              <td>tiling path by array</td>
              <td><a href="/efo/EFO_0001031">EFO_0001031</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3506/">E-MTAB-3506</a></td>
          </tr>
          <tr>
              <td>transcription profiling by array</td>
              <td><a href="/efo/EFO_0002768">EFO_0002768</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2133/">E-MTAB-2133</a></td>
          </tr>
          <tr>
              <td>transcription profiling by MPSS</td>
              <td><a href="/efo/EFO_0002942">EFO_0002942</a></td>
              <td><a href="/arrayexpress/experiments/E-GEOD-50020/">E-GEOD-50020</a></td>
          </tr>
          <tr>
              <td>transcription profiling by RT-PCR</td>
              <td><a href="/efo/EFO_0002943">EFO_0002943</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3152/">E-MTAB-3152</a></td>
          </tr>
          <tr>
              <td>transcription profiling by SAGE</td>
              <td><a href="/efo/EFO_0002941">EFO_0002941</a></td>
              <td><a href="/arrayexpress/experiments/E-GEOD-30069/">E-GEOD-30069</a></td>
          </tr>
          <tr>
              <td>transcription profiling by tiling array</td>
              <td><a href="/efo/EFO_0002769">EFO_0002769</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2241/">E-MTAB-2241</a></td>
          </tr>
          <tr>
              <td>translation profiling</td>
              <td><a href="/efo/EFO_0001033">EFO_0001033</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-3583/">E-MTAB-3583</a></td>
          </tr>
          <tr>
              <td>whole chromosome random sequencing</td>
              <td><a href="/efo/EFO_0004160">EFO_0004160</a></td>
              <td><a href="/arrayexpress/experiments/E-MTAB-2888/">E-MTAB-2888</a></td>
          </tr>
        </tbody>
      </table>
    </div>

    <h3 id="what_to_prepare">What do I need to prepare?</h3>

    <h4>Breadth of the data and metadata</h4>
    <p>The aim is that an ArrayExpress user should have everything they need for the data set to make sense and be reproducible without referring to an associated paper.</p>
    <p>Microarray submissions follow the "Minimum Information About a Microarray Experiment" (<a href="http://fged.org/projects/miame/" target="_blank">MIAME</a>) guidelines. Sequencing submissions follow a similar set of guidelines, "Minimum Information About a Sequencing Experiment" (<a href="http://fged.org/projects/minseqe/" target="_blank">MINSEQE</a>).</p>
    <br>
    <div>
      <img style="display:block; margin-right:auto; margin-left:auto" src="../assets/images/what-to-submit.jpg" alt="What to submit" />
    </div>
    <br>
    <p><strong>As a submitter, you may need to consult with your colleagues, e.g. collaborators or the core facility personnel performing microarray hybridisation or sequencing for you, to gather all detailed information for a successful submission.</strong></p>
    <h4>Metadata</h4>
    <ul class="ae-help_checklist">
      <li class="icon-bullet"><span class="icon icon-common icon-check"></span> Experiment description to give context to the data set (not pasting publication abstract please)</li>
      <li class="icon-bullet"><span class="icon icon-common icon-check"></span> Protocols of all experimental (e.g. sample sourcing, sequencing library preparation) and data analysis  procedures</li>
      <li class="icon-bullet"><span class="icon icon-common icon-check"></span> Sample annotation (as much details as possible, e.g. age of the plant, mouse strain, cell type)</li>
      <li class="icon-bullet"><span class="icon icon-common icon-check"></span> Author information (please include the principal investigator of the project)</li>
      <li class="icon-bullet"><span class="icon icon-common icon-check"></span> <a href="seq_lib_spec.html">Sequencing library specification</a> (NGS experiments only)</li>
    </ul>
    <h4>Raw data</h4>
      <ul class="ae-help_checklist">
        <li class="icon-bullet"><span class="icon icon-common icon-check"></span> Unprocessed data files
        <p class="array"><strong>Microarray</strong>: files obtained from the microarray scanner (e.g. Affymetrix <em>CEL</em> files, Agilent feature extraction <em>txt</em> files, Illumina <em>idat</em> files)<br>
           See the <a href="accepted_raw_ma_file_formats.html">full list of accepted microarray raw data files</a>.</p>
        <p class="ngs"><strong>Sequencing</strong>: compressed raw sequence read files (e.g. <em>fastq.gz</em> files)<br>
          See <a href="uhts_submissions.html#HowToSubmit">accepted sequencing raw data files</a>.</p>
        </li>
      </ul>
    <h4>Processed data</h4>
      <ul class="ae-help_checklist">
        <li class="icon-bullet"><span class="icon icon-common icon-check"></span> Processed data matrix in tab-delimited <em>txt</em> format</li>
        <li class="icon-bullet"><span class="icon icon-common icon-check"></span> Other processed data formats, e.g. <em>bam</em> alignment files (optional)</li>
      </ul>

  </div>

  <!-- side navigation -->
  <div class="small-12 medium-3 columns sidebar">
    <ul class="vertical menu accordion-menu" data-accordion-menu>
      <li><a href="/fg/annotare/help/" class="active" >Submission guide</a></li>
      <li><a>Before you start</a>
        <ul class="menu vertical nested is-active">
          <li><a href="what_to_submit.html" class="active">Submission overview</a>
            <ul class="menu vertical nested is-active">
              <li><a href="what_to_submit.html">What datasets can be submitted?</a></li>
              <li><a href="experiment_types.html">Experiment types in ArrayExpress</a></li>
              <li><a href="what_to_submit.html#what_to_prepare">What do I need to prepare?</a></li>
            </ul>
          </li>
          <li><a href="">Accepted file formats</a>
            <ul>
              <li><a href="accepted_raw_ma_file_formats.html">Microarray raw data</a></li>
              <li><a href="accepted_processed_ma_file_formats.html">Microarray processed data</a></li>
              <li><a href="UHTS_submissions.html#HowToSubmit">Sequencing raw data</a></li>
              <li><a href="UHTS_submissions.html#processed_data">Sequencing processed data</a></li>
            </ul>
          </li>
          <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
          <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
        </ul>
      </li>
      <li><a>Annotare step-by-step</a>
        <ul class="menu vertical nested">
          <li><a href="getting_started.html">Getting started</a></li>
          <li><a href="time_saving_features.html">Time-saving features</a></li>
          <li><a href="describe_exp.html">Describe experiment</a></li>
          <li><a href="sample_attributes.html">Sample attributes and protocols</a></li>
          <li><a href="">Technology-specific details</a>
            <ul>
              <li><a href="seq_lib_spec.html">Sequencing library information</a></li>
              <li><a href="singlecell_submission_guide.html">Single cell information</a></li>
              <li><a href="two_color_ma.html">Two-colour experiments</a></li>
            </ul>
          </li>
          <li><a href="">Upload and assign files</a>
            <ul>
              <li><a href="file_upload.html#direct">Direct upload</a></li>
              <li><a href="ftp_upload.html">FTP/Aspera upload</a></li>
              <li><a href="file_upload.html#ftp">Import FTP/Aspera uploaded files</a></li>
              <li><a href="assign_files.html">Assign files</a></li>
            </ul>
          </li>
          <li><a href="">Validate and submit</a>
            <ul>
              <li><a href="validate_exp.html#validate">What is validation?</a></li>
              <li><a href="submit_exp.html">Submit, and what happens next?</a></li>
              <li><a href="submit_exp.html#save_sub">Saving a copy of your submission</a></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="">After submission</a>
        <ul>
          <li><a href="after_sub_private_access.html">Access private experiments</a></li>
          <li><a href="after_sub_modify_data.html">Modify or remove an existing experiment</a></li>
          <li><a href="after_sub_data_availability.html">Data availability and release policy</a></li>
        </ul>
      </li>
    </ul>
  </div> <!-- /medium-3 -->

</section>
